<?xml version="1.0" encoding="UTF-8"?><Posts><Post><PostId>443813</PostId><Url>http://stackoverflow.com/questions/443813</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\mike\Documents\python\arl xml\turn 24 reader</File><Line>52</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>468383</PostId><Url>http://stackoverflow.com/questions/468383</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>recovery.py</File><Line>8</Line><Method>source = sys.argv[1]</Method></Frame></Stack></Post><Post><PostId>966983</PostId><Url>http://stackoverflow.com/questions/966983</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:/Python26/tcount2.py</File><Line>6</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>998314</PostId><Url>http://stackoverflow.com/questions/998314</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>cube.py</File><Line>5</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>1985178</PostId><Url>http://stackoverflow.com/questions/1985178</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>D:\Python26\PYTHON-PROGRAMME\00000000000000000</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>2211705</PostId><Url>http://stackoverflow.com/questions/2211705</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\zjm_code\sphinx_test\models.py</File><Line>16</Line><Method>module</Method></Frame><Frame><File>D:\Python25\Lib\site-packages\django\db\models\base.py</File><Line>52</Line><Method>__new__</Method></Frame></Stack></Post><Post><PostId>2211816</PostId><Url>http://stackoverflow.com/questions/2211816</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\zjm_code\sphinx_test\books\models.py</File><Line>17</Line><Method>module</Method></Frame><Frame><File>D:\Python25\Lib\site-packages\django\db\models\base.py</File><Line>52</Line><Method>__new__</Method></Frame></Stack></Post><Post><PostId>2364531</PostId><Url>http://stackoverflow.com/questions/2364531</Url><Stack><Type>IndexError</Type><Message>The query returned fewer than 1 results</Message><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\ext\webapp\__init__.py</File><Line>507</Line><Method>__call__</Method></Frame><Frame><File>path\to\src\Main.py</File><Line>42</Line><Method>get</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\ext\db\__init__.py</File><Line>1717</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>2384700</PostId><Url>http://stackoverflow.com/questions/2384700</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>/root/workspace/test/src/test.py</File><Line>61</Line><Method>module</Method></Frame><Frame><File>/root/workspace/test/src/test.py</File><Line>57</Line><Method>main</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/wx-2.8-gtk2-unicode/wx/_core.py</File><Line>7974</Line><Method>__init__</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/wx-2.8-gtk2-unicode/wx/_core.py</File><Line>7548</Line><Method>_BootstrapApp</Method></Frame><Frame><File>/root/workspace/test/src/test.py</File><Line>51</Line><Method>OnInit</Method></Frame><Frame><File>/root/workspace/test/src/test.py</File><Line>39</Line><Method>__init__</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/wx-2.8-gtk2-unicode/wx/lib/agw/hypertreelist.py</File><Line>3282</Line><Method>delegate</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/wx-2.8-gtk2-unicode/wx/lib/agw/hypertreelist.py</File><Line>1414</Line><Method>SetItemWindow</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/wx-2.8-gtk2-unicode/wx/lib/agw/hypertreelist.py</File><Line>1055</Line><Method>SetWindow</Method></Frame></Stack></Post><Post><PostId>2396382</PostId><Url>http://stackoverflow.com/questions/2396382</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Python26/ritaj2.py</File><Line>9</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>2457529</PostId><Url>http://stackoverflow.com/questions/2457529</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>from_list_to_histogram.py</File><Line>43</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>2780579</PostId><Url>http://stackoverflow.com/questions/2780579</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>F:\JavaWorkspace\Test\src\recent_track_database.py</File><Line>50</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>2885627</PostId><Url>http://stackoverflow.com/questions/2885627</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>upload_all_inventory_ebay.py</File><Line>403</Line><Method>module</Method></Frame><Frame><File>upload_all_inventory_ebay.py</File><Line>294</Line><Method>get_item_list</Method></Frame><Frame><File>upload_all_inventory_ebay.py</File><Line>403</Line><Method>module</Method></Frame><Frame><File>upload_all_inventory_ebay.py</File><Line>292</Line><Method>get_item_list</Method></Frame></Stack></Post><Post><PostId>2885627</PostId><Url>http://stackoverflow.com/questions/2885627</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>upload_all_inventory_ebay.py</File><Line>403</Line><Method>module</Method></Frame><Frame><File>upload_all_inventory_ebay.py</File><Line>294</Line><Method>get_item_list</Method></Frame><Frame><File>upload_all_inventory_ebay.py</File><Line>403</Line><Method>module</Method></Frame><Frame><File>upload_all_inventory_ebay.py</File><Line>292</Line><Method>get_item_list</Method></Frame></Stack></Post><Post><PostId>2965436</PostId><Url>http://stackoverflow.com/questions/2965436</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#13</File><Line>2</Line><Method>module</Method></Frame><Frame><File>pyshell#27</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>2965436</PostId><Url>http://stackoverflow.com/questions/2965436</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#13</File><Line>2</Line><Method>module</Method></Frame><Frame><File>pyshell#27</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>3018914</PostId><Url>http://stackoverflow.com/questions/3018914</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/usr/share/pygobject/2.0/codegen/codegen.py</File><Line>1720</Line><Method>module</Method></Frame><Frame><File>/usr/share/pygobject/2.0/codegen/codegen.py</File><Line>1672</Line><Method>main</Method></Frame><Frame><File>/usr/share/pygobject/2.0/codegen/override.py</File><Line>52</Line><Method>__init__</Method></Frame><Frame><File>/usr/share/pygobject/2.0/codegen/override.py</File><Line>84</Line><Method>handle_file</Method></Frame><Frame><File>/usr/share/pygobject/2.0/codegen/override.py</File><Line>96</Line><Method>__parse_override</Method></Frame></Stack></Post><Post><PostId>3269484</PostId><Url>http://stackoverflow.com/questions/3269484</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>string</File><Line>244</Line><Method>run_nodebug</Method></Frame><Frame><File>module1</File><Line>3</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>3396895</PostId><Url>http://stackoverflow.com/questions/3396895</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#8</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:\Python26\read_file_2.py</File><Line>14</Line><Method>process_file</Method></Frame></Stack></Post><Post><PostId>3722771</PostId><Url>http://stackoverflow.com/questions/3722771</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>6</Line><Method>foo</Method></Frame></Stack></Post><Post><PostId>3995563</PostId><Url>http://stackoverflow.com/questions/3995563</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:\dev\snippets\lexical.py</File><Line>92</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>4158030</PostId><Url>http://stackoverflow.com/questions/4158030</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\MYPY\test.py</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>4163257</PostId><Url>http://stackoverflow.com/questions/4163257</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./firstDate</File><Line>4</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>4338732</PostId><Url>http://stackoverflow.com/questions/4338732</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>append_test.py</File><Line>23</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>4619996</PostId><Url>http://stackoverflow.com/questions/4619996</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>html2md.py</File><Line>5</Line><Method>print html2text.html2text(sys.argv a href="http://stackoverflow.com/questions/4686842/how-to-stdin-and-stdout-with-php-and-python-to-use-html2text-and-get-a-markdown-f" 1 /a )</Method></Frame></Stack></Post><Post><PostId>4664502</PostId><Url>http://stackoverflow.com/questions/4664502</Url><Stack><Type>IndexError</Type><Message>0-d arrays can only use a single () or a list of newaxes (and a single ...) as an index</Message><Frame><File>pyshell#3</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>4821795</PostId><Url>http://stackoverflow.com/questions/4821795</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#96</File><Line>1</Line><Method>module</Method></Frame><Frame><File>pyshell#94</File><Line>6</Line><Method>copyAndWriteExcludingPoundSigns_Bad</Method></Frame></Stack></Post><Post><PostId>4866698</PostId><Url>http://stackoverflow.com/questions/4866698</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>parse_stack.py</File><Line>41</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>5174343</PostId><Url>http://stackoverflow.com/questions/5174343</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>filepath etc etc</File><Line>67</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>5308022</PostId><Url>http://stackoverflow.com/questions/5308022</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>E:\my python\Internship\mergestackoverflow.py</File><Line>16</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\xlwt\Workbook.py</File><Line>634</Line><Method>save</Method></Frame><Frame><File>C:\Python27\lib\site-packages\xlwt\Workbook.py</File><Line>615</Line><Method>get_biff_data</Method></Frame></Stack></Post><Post><PostId>5346144</PostId><Url>http://stackoverflow.com/questions/5346144</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\lib\site-packages\imdb\parser\http\utils.py</File><Line>555</Line><Method>xpath</Method></Frame><Frame><File>C:\Python27\lib\site-packages\imdb\parser\http\bsouplxml\etree.py</File><Line>57</Line><Method>xpath</Method></Frame><Frame><File>C:\Python27\lib\site-packages\imdb\parser\http\bsouplxml\bsoupxpath.py</File><Line>113</Line><Method>apply</Method></Frame><Frame><File>C:\Python27\lib\site-packages\imdb\parser\http\bsouplxml\bsoupxpath.py</File><Line>287</Line><Method>apply</Method></Frame><Frame><File>C:\Python27\lib\site-packages\imdb\parser\http\bsouplxml\bsoupxpath.py</File><Line>331</Line><Method>__call__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\imdb\parser\http\bsouplxml\bsoupxpath.py</File><Line>360</Line><Method>__starts_with</Method></Frame></Stack></Post><Post><PostId>5511715</PostId><Url>http://stackoverflow.com/questions/5511715</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\anlopes\workspace\redes_sociais\src\twitterC.py</File><Line>77</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>5537618</PostId><Url>http://stackoverflow.com/questions/5537618</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#21</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>5609554</PostId><Url>http://stackoverflow.com/questions/5609554</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\lib\lib-tk\Tkinter.py</File><Line>1410</Line><Method>strong call /strong</Method></Frame><Frame><File>C:\Users\Psao\Desktop\whalewhire\WhaleWire.py</File><Line>93</Line><Method>whale</Method></Frame></Stack></Post><Post><PostId>5708352</PostId><Url>http://stackoverflow.com/questions/5708352</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#121</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:\Python26\lib\site-packages\nltk\tree.py</File><Line>139</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>5771081</PostId><Url>http://stackoverflow.com/questions/5771081</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/__init__.py</File><Line>634</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/apps/spare-wheels/1.349945145699425513/sparewheels.py</File><Line>92</Line><Method>get</Method></Frame><Frame><File>/base/data/home/apps/spare-wheels/1.349945145699425513/sparewheels.py</File><Line>196</Line><Method>get_secure</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/__init__.py</File><Line>634</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/apps/spare-wheels/1.349947004526942195/sparewheels.py</File><Line>92</Line><Method>get</Method></Frame><Frame><File>/base/data/home/apps/spare-wheels/1.349947004526942195/sparewheels.py</File><Line>196</Line><Method>get_secure</Method></Frame></Stack></Post><Post><PostId>5771081</PostId><Url>http://stackoverflow.com/questions/5771081</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/__init__.py</File><Line>634</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/apps/spare-wheels/1.349945145699425513/sparewheels.py</File><Line>92</Line><Method>get</Method></Frame><Frame><File>/base/data/home/apps/spare-wheels/1.349945145699425513/sparewheels.py</File><Line>196</Line><Method>get_secure</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/__init__.py</File><Line>634</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/apps/spare-wheels/1.349947004526942195/sparewheels.py</File><Line>92</Line><Method>get</Method></Frame><Frame><File>/base/data/home/apps/spare-wheels/1.349947004526942195/sparewheels.py</File><Line>196</Line><Method>get_secure</Method></Frame></Stack></Post><Post><PostId>5812817</PostId><Url>http://stackoverflow.com/questions/5812817</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>c.py</File><Line>18</Line><Method>module</Method></Frame><Frame><File>c.py</File><Line>12</Line><Method>b</Method></Frame></Stack></Post><Post><PostId>6288222</PostId><Url>http://stackoverflow.com/questions/6288222</Url><Stack><Type>IndexError</Type><Message>index out of range for array</Message><Frame><File>main02.py</File><Line>66</Line><Method>module</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/axes.py</File><Line>3788</Line><Method>plot</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/axes.py</File><Line>1824</Line><Method>autoscale_view</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/ticker.py</File><Line>1170</Line><Method>view_limits</Method></Frame><Frame><File>/film/tools/PythonExtensions/v41/py26_linux-x64/numpy/core/fromnumeric.py</File><Line>103</Line><Method>take</Method></Frame></Stack></Post><Post><PostId>6313943</PostId><Url>http://stackoverflow.com/questions/6313943</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>rsa.py</File><Line>94</Line><Method>print dec_rsa(b, d, n)</Method></Frame><Frame><File>rsa.py</File><Line>88</Line><Method>dec_rsa</Method></Frame><Frame><File>rsa.py</File><Line>77</Line><Method>decrypt_byte</Method></Frame><Frame><File>rsa.py</File><Line>67</Line><Method>find_key</Method></Frame></Stack></Post><Post><PostId>6478120</PostId><Url>http://stackoverflow.com/questions/6478120</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\sample.py</File><Line>7</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>6556347</PostId><Url>http://stackoverflow.com/questions/6556347</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/mo/Projects/mazban/lib/python2.6/site-packages/fabric/main.py</File><Line>723</Line><Method>main</Method></Frame><Frame><File>/home/mo/Projects/mazban/mazban/fabfile.py</File><Line>203</Line><Method>firstrun</Method></Frame></Stack></Post><Post><PostId>6860304</PostId><Url>http://stackoverflow.com/questions/6860304</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>CNV_length_cal.py</File><Line>8</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>6988557</PostId><Url>http://stackoverflow.com/questions/6988557</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/__init__.py</File><Line>702</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/apps/s~testtest/1.352363227571120815/main.py</File><Line>62</Line><Method>post</Method></Frame></Stack></Post><Post><PostId>7003821</PostId><Url>http://stackoverflow.com/questions/7003821</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:/Python27/forListCreateTest</File><Line>21</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>7144571</PostId><Url>http://stackoverflow.com/questions/7144571</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\EclipseWorkspaces\csse120\python\src\Isnin2.py</File><Line>20</Line><Method>test_isnin2</Method></Frame><Frame><File>C:\Program Files\Python27\lib\site-packages\selenium-2.4.0-py2.7.egg\selenium\webdriver\remote\webdriver.py</File><Line>154</Line><Method>get</Method></Frame><Frame><File>C:\Program Files\Python27\lib\site-packages\selenium-2.4.0-py2.7.egg\selenium\webdriver\remote\webdriver.py</File><Line>144</Line><Method>execute</Method></Frame><Frame><File>C:\Program Files\Python27\lib\site-packages\selenium-2.4.0-py2.7.egg\selenium\webdriver\remote\errorhandler.py</File><Line>111</Line><Method>check_response</Method></Frame></Stack></Post><Post><PostId>7169348</PostId><Url>http://stackoverflow.com/questions/7169348</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>table.py</File><Line>45</Line><Method>module</Method></Frame><Frame><File>table.py</File><Line>24</Line><Method>width</Method></Frame></Stack></Post><Post><PostId>7177285</PostId><Url>http://stackoverflow.com/questions/7177285</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>./fastq_phred_filter.py</File><Line>24</Line><Method>?</Method></Frame></Stack></Post><Post><PostId>7184796</PostId><Url>http://stackoverflow.com/questions/7184796</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#31</File><Line>6</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>7445157</PostId><Url>http://stackoverflow.com/questions/7445157</Url><Stack><Type>IndexError</Type><Message>arrays used as indices must be of integer (or boolean) type</Message><Frame><File>qstk1.py</File><Line>37</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>7489956</PostId><Url>http://stackoverflow.com/questions/7489956</Url><Stack><Type>IndexError</Type><Message>0-d arrays can't be indexed.</Message><Frame><File>ipython-input-19-ed41131430b9</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>7523530</PostId><Url>http://stackoverflow.com/questions/7523530</Url><Stack><Type>IndexError</Type><Message>0-d arrays can't be indexed</Message><Frame><File>pyshell#98</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>7641777</PostId><Url>http://stackoverflow.com/questions/7641777</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Program Files (x86)\Google\google_appengine\google\appengine\ext\webapp\_webapp25.py</File><Line>701</Line><Method>__call__</Method></Frame><Frame><File>C:\U\A\D\\toplawfirms.py</File><Line>384</Line><Method>get</Method></Frame></Stack></Post><Post><PostId>7762309</PostId><Url>http://stackoverflow.com/questions/7762309</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>asdf.py</File><Line>6</Line><Method>module</Method></Frame><Frame><File>asdf.py</File><Line>6</Line><Method>genexpr</Method></Frame></Stack></Post><Post><PostId>7853606</PostId><Url>http://stackoverflow.com/questions/7853606</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python32\Sample Program\csvParser.py</File><Line>9</Line><Method>module</Method></Frame><Frame><File>C:\Python32\Sample Program\csvParser.py</File><Line>13</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>7853606</PostId><Url>http://stackoverflow.com/questions/7853606</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python32\Sample Program\csvParser.py</File><Line>9</Line><Method>module</Method></Frame><Frame><File>C:\Python32\Sample Program\csvParser.py</File><Line>13</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>7897839</PostId><Url>http://stackoverflow.com/questions/7897839</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./clean.py</File><Line>8</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>7952853</PostId><Url>http://stackoverflow.com/questions/7952853</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\test.py</File><Line>112</Line><Method>get_ntlm_data</Method></Frame><Frame><File>C:\Python27\lib\urllib2.py</File><Line>126</Line><Method>urlopen</Method></Frame><Frame><File>C:\Python27\lib\urllib2.py</File><Line>398</Line><Method>open</Method></Frame><Frame><File>C:\Python27\lib\urllib2.py</File><Line>511</Line><Method>http_response</Method></Frame><Frame><File>C:\Python27\lib\urllib2.py</File><Line>430</Line><Method>error</Method></Frame><Frame><File>C:\Python27\lib\urllib2.py</File><Line>370</Line><Method>_call_chain</Method></Frame><Frame><File>C:\Python27\lib\site-packages\python_ntlm-1.0.1-py2.7.egg\ntlm\HTTPNtlmAuthHandler.py</File><Line>99</Line><Method>http_error_401</Method></Frame><Frame><File>C:\Python27\lib\site-packages\python_ntlm-1.0.1-py2.7.egg\ntlm\HTTPNtlmAuthHandler.py</File><Line>35</Line><Method>http_error_authentication_required</Method></Frame><Frame><File>C:\Python27\lib\site-packages\python_ntlm-1.0.1-py2.7.egg\ntlm\HTTPNtlmAuthHandler.py</File><Line>72</Line><Method>retry_using_http_NTLM_auth</Method></Frame></Stack></Post><Post><PostId>8245660</PostId><Url>http://stackoverflow.com/questions/8245660</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>Network_property.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>8322561</PostId><Url>http://stackoverflow.com/questions/8322561</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>C:/Users/xxxxxxxx/Desktop/t.py</File><Line>44</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>8452285</PostId><Url>http://stackoverflow.com/questions/8452285</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/dev_appserver.py</File><Line>4039</Line><Method>_HandleRequest</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/dev_appserver.py</File><Line>3945</Line><Method>_Dispatch</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/dev_appserver.py</File><Line>605</Line><Method>Dispatch</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/dev_appserver.py</File><Line>3015</Line><Method>Dispatch</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/dev_appserver.py</File><Line>2925</Line><Method>ExecuteCGI</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/dev_appserver.py</File><Line>2787</Line><Method>ExecuteOrImportScript</Method></Frame><Frame><File>/Users/bryce/Documents/Aptana Studio 3 Workspace/DennysBE/djangoappengine/main/main.py</File><Line>87</Line><Method>real_main</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/ext/webapp/util.py</File><Line>98</Line><Method>run_wsgi_app</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/ext/webapp/util.py</File><Line>116</Line><Method>run_bare_wsgi_app</Method></Frame><Frame><File>/Users/bryce/Documents/Aptana Studio 3 Workspace/DennysBE/django/core/handlers/wsgi.py</File><Line>273</Line><Method>__call__</Method></Frame><Frame><File>/Users/bryce/Documents/Aptana Studio 3 Workspace/DennysBE/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/Users/bryce/Documents/Aptana Studio 3 Workspace/DennysBE/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/Users/bryce/Documents/Aptana Studio 3 Workspace/DennysBE/django/views/debug.py</File><Line>59</Line><Method>technical_500_response</Method></Frame><Frame><File>/Users/bryce/Documents/Aptana Studio 3 Workspace/DennysBE/django/views/debug.py</File><Line>114</Line><Method>get_traceback_html</Method></Frame><Frame><File>/Users/bryce/Documents/Aptana Studio 3 Workspace/DennysBE/django/views/debug.py</File><Line>242</Line><Method>get_traceback_frames</Method></Frame><Frame><File>/Users/bryce/Documents/Aptana Studio 3 Workspace/DennysBE/django/views/debug.py</File><Line>223</Line><Method>_get_lines_from_file</Method></Frame></Stack></Post><Post><PostId>8601446</PostId><Url>http://stackoverflow.com/questions/8601446</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/usr/local/lib/python2.7/dist-packages/django/core/servers/basehttp.py</File><Line>283</Line><Method>run</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/contrib/staticfiles/handlers.py</File><Line>68</Line><Method>__call__</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/core/handlers/wsgi.py</File><Line>273</Line><Method>__call__</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/views/debug.py</File><Line>59</Line><Method>technical_500_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/views/debug.py</File><Line>114</Line><Method>get_traceback_html</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/views/debug.py</File><Line>242</Line><Method>get_traceback_frames</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/views/debug.py</File><Line>223</Line><Method>_get_lines_from_file</Method></Frame></Stack></Post><Post><PostId>8665496</PostId><Url>http://stackoverflow.com/questions/8665496</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>dasdsad.py</File><Line>35</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>8714093</PostId><Url>http://stackoverflow.com/questions/8714093</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\phenny\bot.py</File><Line>189</Line><Method>call</Method></Frame><Frame><File>C:\phenny\modules\isup.py</File><Line>18</Line><Method>isup</Method></Frame></Stack></Post><Post><PostId>8737430</PostId><Url>http://stackoverflow.com/questions/8737430</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\ctee1\Desktop\pyparsing\outputparser.py</File><Line>39</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>8796593</PostId><Url>http://stackoverflow.com/questions/8796593</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>pyshell#3</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>8850384</PostId><Url>http://stackoverflow.com/questions/8850384</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/Users/me/Documents/Programming/python/eclipse/workspace/concat.py</File><Line>8</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>8950151</PostId><Url>http://stackoverflow.com/questions/8950151</Url><Stack><Type>IndexError</Type><Message>list index out of rang</Message><Frame><File>./SocialMonitor.py</File><Line>39</Line><Method>module</Method></Frame><Frame><File>./SocialMonitor.py</File><Line>40</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>8950151</PostId><Url>http://stackoverflow.com/questions/8950151</Url><Stack><Type>IndexError</Type><Message>list index out of rang</Message><Frame><File>./SocialMonitor.py</File><Line>39</Line><Method>module</Method></Frame><Frame><File>./SocialMonitor.py</File><Line>40</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>9138901</PostId><Url>http://stackoverflow.com/questions/9138901</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>superrogue.py</File><Line>85</Line><Method>module</Method></Frame><Frame><File>superrogue.py</File><Line>68</Line><Method>make_map</Method></Frame></Stack></Post><Post><PostId>9249406</PostId><Url>http://stackoverflow.com/questions/9249406</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>11</Line><Method>group_ldata</Method></Frame></Stack></Post><Post><PostId>9437162</PostId><Url>http://stackoverflow.com/questions/9437162</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\tao\Dropbox\AppPest1\trex\trex_input.py</File><Line>33</Line><Method>module</Method></Frame><Frame><File>C:\Program Files (x86)\Google\google_appengine\lib\django_1_2\django\db\models\base.py</File><Line>50</Line><Method>__new__</Method></Frame></Stack></Post><Post><PostId>9471594</PostId><Url>http://stackoverflow.com/questions/9471594</Url><Stack><Type>IndexError</Type><Message>invalid index</Message><Frame><File>my main file</File><Line>152</Line><Method>module</Method></Frame><Frame><File>my main file</File><Line>148</Line><Method>main</Method></Frame><Frame><File>my main file</File><Line>125</Line><Method>run</Method></Frame><Frame><File>virtualenv \Lib\site-packages\pyglet\app\__init__.py</File><Line>123</Line><Method>run</Method></Frame><Frame><File>virtualenv \Lib\site-packages\pyglet\app\base.py</File><Line>135</Line><Method>run</Method></Frame><Frame><File>virtualenv \Lib\site-packages\pyglet\app\base.py</File><Line>164</Line><Method>_run_estimated</Method></Frame><Frame><File>virtualenv \Lib\site-packages\pyglet\app\base.py</File><Line>278</Line><Method>idle</Method></Frame><Frame><File>virtualenv \Lib\site-packages\pyglet\window\win32\__init__.py</File><Line>305</Line><Method>switch_to</Method></Frame><Frame><File>virtualenv \Lib\site-packages\pyglet\gl\win32.py</File><Line>213</Line><Method>set_current</Method></Frame><Frame><File>virtualenv \Lib\site-packages\pyglet\gl\base.py</File><Line>320</Line><Method>set_current</Method></Frame></Stack></Post><Post><PostId>9472932</PostId><Url>http://stackoverflow.com/questions/9472932</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>iterating.py</File><Line>37</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>9542682</PostId><Url>http://stackoverflow.com/questions/9542682</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Documents and Settings\My Documents\Symplectic Integrators\RK4_2.py</File><Line>23</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>9747158</PostId><Url>http://stackoverflow.com/questions/9747158</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\lib\site-packages\spyderlib\spyder.py</File><Line>1547</Line><Method>main</Method></Frame><Frame><File>C:\Python27\lib\site-packages\spyderlib\spyder.py</File><Line>1472</Line><Method>run_spyder</Method></Frame><Frame><File>C:\Python27\lib\site-packages\spyderlib\spyder.py</File><Line>555</Line><Method>setup</Method></Frame><Frame><File>C:\Python27\lib\site-packages\spyderlib\plugins\console.py</File><Line>54</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\spyderlib\widgets\internalshell.py</File><Line>101</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\spyderlib\widgets\shell.py</File><Line>698</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\spyderlib\widgets\shell.py</File><Line>61</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\spyderlib\widgets\shell.py</File><Line>500</Line><Method>load_history</Method></Frame></Stack></Post><Post><PostId>9816139</PostId><Url>http://stackoverflow.com/questions/9816139</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>merged-rrd.py</File><Line>149</Line><Method>module</Method></Frame><Frame><File>merged-rrd.py</File><Line>77</Line><Method>mergeRRD</Method></Frame><Frame><File>merged-rrd.py</File><Line>52</Line><Method>getXmlDict</Method></Frame></Stack></Post><Post><PostId>9857352</PostId><Url>http://stackoverflow.com/questions/9857352</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\GVRSQA004\Desktop\udCombo.py</File><Line>86</Line><Method>fileFormats</Method></Frame></Stack></Post><Post><PostId>9881353</PostId><Url>http://stackoverflow.com/questions/9881353</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./xyz.py</File><Line>42</Line><Method>g</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery-2.4.6-py2.7.egg/celery/result.py</File><Line>99</Line><Method>wait</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery-2.4.6-py2.7.egg/celery/result.py</File><Line>95</Line><Method>get</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery-2.4.6-py2.7.egg/celery/backends/amqp.py</File><Line>147</Line><Method>wait_for</Method></Frame></Stack></Post><Post><PostId>9935271</PostId><Url>http://stackoverflow.com/questions/9935271</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>/home/abc/Desktop/Return.py</File><Line>6</Line><Method>module</Method></Frame><Frame><File>/usr/lib/python2.6/dist-packages/numpy/linalg/linalg.py</File><Line>355</Line><Method>inv</Method></Frame></Stack></Post><Post><PostId>10119819</PostId><Url>http://stackoverflow.com/questions/10119819</Url><Stack><Type>IndexError</Type><Message>pop from empty list</Message><Frame><File>/usr/lib/python2.4/site-packages/multiprocessing-2.6.2.1-py2.4-linux-i686.egg/multiprocessing/process.py</File><Line>237</Line><Method>_bootstrap</Method></Frame><Frame><File>/usr/lib/python2.4/site-packages/multiprocessing-2.6.2.1-py2.4-linux-i686.egg/multiprocessing/process.py</File><Line>93</Line><Method>run</Method></Frame><Frame><File>test.py</File><Line>419</Line><Method>func_b</Method></Frame></Stack></Post><Post><PostId>10143600</PostId><Url>http://stackoverflow.com/questions/10143600</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/Library/Frameworks/Python.framework/Versions/2.7/bin/git-review</File><Line>676</Line><Method>module</Method></Frame><Frame><File>/Library/Frameworks/Python.framework/Versions/2.7/bin/git-review</File><Line>643</Line><Method>main</Method></Frame><Frame><File>/Library/Frameworks/Python.framework/Versions/2.7/bin/git-review</File><Line>122</Line><Method>set_hooks_commit_msg</Method></Frame><Frame><File>/Library/Frameworks/Python.framework/Versions/2.7/bin/git-review</File><Line>247</Line><Method>parse_git_show</Method></Frame></Stack></Post><Post><PostId>10210062</PostId><Url>http://stackoverflow.com/questions/10210062</Url><Stack><Type>IndexError</Type><Message>list index out of range'</Message><Frame><File>./test.py</File><Line>14</Line><Method>?</Method></Frame></Stack></Post><Post><PostId>10220323</PostId><Url>http://stackoverflow.com/questions/10220323</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\name\Desktop\game\game.py</File><Line>37</Line><Method>module</Method></Frame><Frame><File>C:\Users\name\Desktop\game\game.py</File><Line>21</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>10368259</PostId><Url>http://stackoverflow.com/questions/10368259</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>poly.py</File><Line>31</Line><Method>__add__</Method></Frame></Stack></Post><Post><PostId>10658271</PostId><Url>http://stackoverflow.com/questions/10658271</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>plotter.py</File><Line>16</Line><Method>module</Method></Frame><Frame><File>plotter.py</File><Line>14</Line><Method>getColumn</Method></Frame></Stack></Post><Post><PostId>10925448</PostId><Url>http://stackoverflow.com/questions/10925448</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#6</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>10925639</PostId><Url>http://stackoverflow.com/questions/10925639</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#16</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>10986308</PostId><Url>http://stackoverflow.com/questions/10986308</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>gettinginput.py</File><Line>25</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>11138062</PostId><Url>http://stackoverflow.com/questions/11138062</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:/Python27/packagings.py</File><Line>47</Line><Method>module</Method></Frame><Frame><File>C:/Python27/packagings.py</File><Line>27</Line><Method>ply_read_values</Method></Frame></Stack></Post><Post><PostId>11217358</PostId><Url>http://stackoverflow.com/questions/11217358</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>stats.py</File><Line>13</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>11270895</PostId><Url>http://stackoverflow.com/questions/11270895</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Joseph\Documents\Programming fun\computer guessing a number game.py</File><Line>32</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\random.py</File><Line>274</Line><Method>choice</Method></Frame></Stack></Post><Post><PostId>11388180</PostId><Url>http://stackoverflow.com/questions/11388180</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>pyshell#41</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>11435209</PostId><Url>http://stackoverflow.com/questions/11435209</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./tdseq4.py</File><Line>124</Line><Method>module</Method></Frame><Frame><File>./tdseq4.py</File><Line>89</Line><Method>main</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/pyplot.py</File><Line>343</Line><Method>figure</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py</File><Line>80</Line><Method>new_figure_manager</Method></Frame><Frame><File>/usr/lib/python2.7/lib-tk/Tkinter.py</File><Line>1683</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>11458902</PostId><Url>http://stackoverflow.com/questions/11458902</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>c.py</File><Line>37</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>11494596</PostId><Url>http://stackoverflow.com/questions/11494596</Url><Stack><Type>IndexError</Type><Message>bytearray index out of range</Message><Frame><File>./decode.py</File><Line>36</Line><Method>module</Method></Frame><Frame><File>./decode.py</File><Line>34</Line><Method>process_packets</Method></Frame><Frame><File>./decode.py</File><Line>22</Line><Method>bin_four_byte_xor</Method></Frame></Stack></Post><Post><PostId>11539938</PostId><Url>http://stackoverflow.com/questions/11539938</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>StressTestCompare.py</File><Line>45</Line><Method>print data[2][0]</Method></Frame></Stack></Post><Post><PostId>11549486</PostId><Url>http://stackoverflow.com/questions/11549486</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>regress.py</File><Line>8</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>11766782</PostId><Url>http://stackoverflow.com/questions/11766782</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>C:\Documents and Settings\My Documents\Computational Physics\Wave-eqn.py</File><Line>56</Line><Method>module</Method></Frame><Frame><File>C:\Python32\lib\site-packages\mpl_toolkits\mplot3d\axes3d.py</File><Line>906</Line><Method>plot_wireframe</Method></Frame><Frame><File>C:\Python32\lib\site-packages\mpl_toolkits\mplot3d\axes3d.py</File><Line>906</Line><Method>listcomp</Method></Frame></Stack></Post><Post><PostId>11770604</PostId><Url>http://stackoverflow.com/questions/11770604</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>plot_data.py</File><Line>286</Line><Method>?</Method></Frame></Stack></Post><Post><PostId>11977060</PostId><Url>http://stackoverflow.com/questions/11977060</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Pulkit\Desktop\Random\nth_prime.py</File><Line>30</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>12009702</PostId><Url>http://stackoverflow.com/questions/12009702</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>grub.py</File><Line>59</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>12014609</PostId><Url>http://stackoverflow.com/questions/12014609</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/base/python27_runtime/python27_lib/versions/third_party/webapp2-2.3/webapp2.py</File><Line>1511</Line><Method>__call__</Method></Frame><Frame><File>/base/python27_runtime/python27_lib/versions/third_party/webapp2-2.3/webapp2.py</File><Line>1505</Line><Method>__call__</Method></Frame><Frame><File>/base/python27_runtime/python27_lib/versions/third_party/webapp2-2.3/webapp2.py</File><Line>1253</Line><Method>default_dispatcher</Method></Frame><Frame><File>/base/python27_runtime/python27_lib/versions/third_party/webapp2-2.3/webapp2.py</File><Line>1077</Line><Method>__call__</Method></Frame><Frame><File>/base/python27_runtime/python27_lib/versions/third_party/webapp2-2.3/webapp2.py</File><Line>547</Line><Method>dispatch</Method></Frame><Frame><File>/base/python27_runtime/python27_lib/versions/third_party/webapp2-2.3/webapp2.py</File><Line>545</Line><Method>dispatch</Method></Frame><Frame><File>myapp.py</File><Line>22</Line><Method>post</Method></Frame></Stack></Post><Post><PostId>12022620</PostId><Url>http://stackoverflow.com/questions/12022620</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>Python_progs/voc.py</File><Line>43</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>12118765</PostId><Url>http://stackoverflow.com/questions/12118765</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>test.py</File><Line>17</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>12144681</PostId><Url>http://stackoverflow.com/questions/12144681</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/tileone/venv/lib/python2.6/site-packages/gunicorn/workers/sync.py</File><Line>93</Line><Method>handle_request</Method></Frame><Frame><File>/home/tileone/venv/lib/python2.6/site-packages/gunicorn/http/wsgi.py</File><Line>146</Line><Method>create</Method></Frame></Stack></Post><Post><PostId>12154958</PostId><Url>http://stackoverflow.com/questions/12154958</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\pythonPractice\firstPython.py</File><Line>13</Line><Method>module</Method></Frame><Frame><File>D:\pythonPractice\firstPython.py</File><Line>9</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>12181129</PostId><Url>http://stackoverflow.com/questions/12181129</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\python\src\SVN_Email_copy.py</File><Line>141</Line><Method>module</Method></Frame><Frame><File>C:\python\src\SVN_Email_copy.py</File><Line>50</Line><Method>xml_data</Method></Frame></Stack></Post><Post><PostId>12193471</PostId><Url>http://stackoverflow.com/questions/12193471</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>49.py</File><Line>9</Line><Method>module</Method></Frame><Frame><File>49.py</File><Line>4</Line><Method>genSet</Method></Frame></Stack></Post><Post><PostId>12322065</PostId><Url>http://stackoverflow.com/questions/12322065</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/mnt/www-data/run/backend/lib/python2.7/site-packages/flask/app.py</File><Line>1504</Line><Method>wsgi_app</Method></Frame><Frame><File>/mnt/www-data/run/backend/lib/python2.7/site-packages/flask/app.py</File><Line>1264</Line><Method>full_dispatch_request</Method></Frame><Frame><File>/mnt/www-data/run/backend/lib/python2.7/site-packages/flask/app.py</File><Line>1262</Line><Method>full_dispatch_request</Method></Frame><Frame><File>/mnt/www-data/run/backend/lib/python2.7/site-packages/flask/app.py</File><Line>1248</Line><Method>dispatch_request</Method></Frame><Frame><File>/mnt/www-data/run/backend/lib/python2.7/site-packages/newrelic-1.4.0.137-py2.7-linux-x86_64.egg/newrelic/api/object_wrapper.py</File><Line>166</Line><Method>__call__</Method></Frame><Frame><File>/mnt/www-data/run/backend/lib/python2.7/site-packages/newrelic-1.4.0.137-py2.7-linux-x86_64.egg/newrelic/api/function_trace.py</File><Line>81</Line><Method>literal_wrapper</Method></Frame><Frame><File>/mnt/www-data/run/backend/lib/python2.7/site-packages/newrelic-1.4.0.137-py2.7-linux-x86_64.egg/newrelic/api/name_transaction.py</File><Line>58</Line><Method>__call__</Method></Frame><Frame><File>/mnt/www-data/run/backend/webservice/decorators/controller_wrapper.py</File><Line>75</Line><Method>decorated_function</Method></Frame><Frame><File>/mnt/www-data/run/backend/webservice/decorators/controller_wrapper.py</File><Line>90</Line><Method>decorated_function</Method></Frame><Frame><File>/mnt/www-data/run/backend/webservice/decorators/login_required.py</File><Line>16</Line><Method>decorated_function</Method></Frame><Frame><File>/mnt/www-data/run/backend/webservice/controllers/me/photos.py</File><Line>28</Line><Method>create_photo</Method></Frame><Frame><File>/mnt/www-data/run/backend/evertale/tasks/photos.py</File><Line>106</Line><Method>store_photos</Method></Frame></Stack></Post><Post><PostId>12483930</PostId><Url>http://stackoverflow.com/questions/12483930</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./sample_script.py</File><Line>111</Line><Method>module</Method></Frame><Frame><File>./sample_script.py</File><Line>108</Line><Method>main</Method></Frame><Frame><File>./sample_script.py</File><Line>100</Line><Method>set_migration_data</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/sqlalchemy/orm/query.py</File><Line>1267</Line><Method>all</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/sqlalchemy/orm/query.py</File><Line>1361</Line><Method>__iter__</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/sqlalchemy/orm/query.py</File><Line>1364</Line><Method>_execute_and_instances</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/sqlalchemy/orm/query.py</File><Line>251</Line><Method>_mapper_zero_or_none</Method></Frame></Stack></Post><Post><PostId>12701428</PostId><Url>http://stackoverflow.com/questions/12701428</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>test.py</File><Line>155</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>12754698</PostId><Url>http://stackoverflow.com/questions/12754698</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>lodarena.py</File><Line>398</Line><Method>module</Method></Frame><Frame><File>lodarena.py</File><Line>393</Line><Method>character_gen</Method></Frame><Frame><File>lodarena.py</File><Line>139</Line><Method>buy_weapon</Method></Frame></Stack></Post><Post><PostId>12760265</PostId><Url>http://stackoverflow.com/questions/12760265</Url><Stack><Type>IndexError</Type><Message>no such group</Message><Frame><File>pyshell#29</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13039392</PostId><Url>http://stackoverflow.com/questions/13039392</Url><Stack><Type>IndexError</Type><Message>list index out of range*</Message><Frame><File>C:\Python32\fantasy.py</File><Line>72</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13145240</PostId><Url>http://stackoverflow.com/questions/13145240</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyprf.py</File><Line>376</Line><Method>module</Method></Frame><Frame><File>pyprf.py</File><Line>358</Line><Method>processPDF</Method></Frame><Frame><File>pyprf.py</File><Line>223</Line><Method>processPage</Method></Frame></Stack></Post><Post><PostId>13150520</PostId><Url>http://stackoverflow.com/questions/13150520</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>\\minerfiles.mst.edu\dfs\users\asadmb\Desktop\Programming 3\Knapsack_Main.py</File><Line>10</Line><Method>module</Method></Frame><Frame><File>\\minerfiles.mst.edu\dfs\users\asadmb\Desktop\Programming 3\Knapsack_Test.py</File><Line>31</Line><Method>greedy_knapsack_test</Method></Frame><Frame><File>\\minerfiles.mst.edu\dfs\users\asadmb\Desktop\Programming 3\KnapsackClass.py</File><Line>30</Line><Method>greedy_knap</Method></Frame></Stack></Post><Post><PostId>13166940</PostId><Url>http://stackoverflow.com/questions/13166940</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/Library/Python/2.7/site-packages/Flask-0.9-py2.7.egg/flask/app.py</File><Line>1701</Line><Method>__call__</Method></Frame><Frame><File>/Library/Python/2.7/site-packages/Flask-0.9-py2.7.egg/flask/app.py</File><Line>1689</Line><Method>wsgi_app</Method></Frame><Frame><File>/Library/Python/2.7/site-packages/Flask-0.9-py2.7.egg/flask/app.py</File><Line>1687</Line><Method>wsgi_app</Method></Frame><Frame><File>/Library/Python/2.7/site-packages/Flask-0.9-py2.7.egg/flask/app.py</File><Line>1360</Line><Method>full_dispatch_request</Method></Frame><Frame><File>/Library/Python/2.7/site-packages/Flask-0.9-py2.7.egg/flask/app.py</File><Line>1358</Line><Method>full_dispatch_request</Method></Frame><Frame><File>/Library/Python/2.7/site-packages/Flask-0.9-py2.7.egg/flask/app.py</File><Line>1344</Line><Method>dispatch_request</Method></Frame><Frame><File>/Users/Zach/Dropbox/bitly/bit.py</File><Line>35</Line><Method>settings</Method></Frame></Stack></Post><Post><PostId>13189424</PostId><Url>http://stackoverflow.com/questions/13189424</Url><Stack><Type>IndexError</Type><Message>tuple index out of range`</Message><Frame><File>/Users/ad9075/PycharmProjects/bednmf/test.py</File><Line>14</Line><Method>testNmfRun</Method></Frame><Frame><File>/Users/ad9075/PycharmProjects/bednmf/nmf.py</File><Line>18</Line><Method>factorization</Method></Frame><Frame><File>/Library/Python/2.7/site-packages/scipy/linalg/basic.py</File><Line>540</Line><Method>pinv</Method></Frame></Stack></Post><Post><PostId>13189527</PostId><Url>http://stackoverflow.com/questions/13189527</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>/home/omar/PycharmProjects/Python_The_Hardway/ex26_Quiz.py</File><Line>76</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13280656</PostId><Url>http://stackoverflow.com/questions/13280656</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Users/75IV740906/Desktop/py117</File><Line>26</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13458430</PostId><Url>http://stackoverflow.com/questions/13458430</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Users/Shane/Documents/Amer CISC/readandmerge.py</File><Line>30</Line><Method>module</Method></Frame><Frame><File>C:/Users/Shane/Documents/Amer CISC/readandmerge.py</File><Line>23</Line><Method>ReadAndMerge</Method></Frame><Frame><File>C:/Users/Shane/Documents/Amer CISC/readandmerge.py</File><Line>23</Line><Method>lambda</Method></Frame></Stack></Post><Post><PostId>13488249</PostId><Url>http://stackoverflow.com/questions/13488249</Url><Stack><Type>IndexError</Type><Message>index out of range</Message><Frame><File>F:\WORK\SOLVEUR\ALGOCODE\PYTHON_\DataSets\DataSets_DONLP2_gaussians.py</File><Line>167</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\matplotlib\pyplot.py</File><Line>766</Line><Method>subplot</Method></Frame><Frame><File>C:\Python27\lib\site-packages\matplotlib\figure.py</File><Line>777</Line><Method>add_subplot</Method></Frame><Frame><File>C:\Python27\lib\site-packages\matplotlib\axes.py</File><Line>8364</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\matplotlib\gridspec.py</File><Line>175</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>13537595</PostId><Url>http://stackoverflow.com/questions/13537595</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#206</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13542720</PostId><Url>http://stackoverflow.com/questions/13542720</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>/Users/Toly/Downloads/ps2 6/ps2_hangman.py</File><Line>82</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13569478</PostId><Url>http://stackoverflow.com/questions/13569478</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#2</File><Line>1</Line><Method>module</Method></Frame><Frame><File>D:\my_stuff\Google Drive\Modules\factors.py</File><Line>50</Line><Method>pFactors</Method></Frame><Frame><File>pyshell#3</File><Line>1</Line><Method>module</Method></Frame><Frame><File>D:\my_stuff\Google Drive\Modules\factors.py</File><Line>50</Line><Method>pFactors</Method></Frame></Stack></Post><Post><PostId>13569478</PostId><Url>http://stackoverflow.com/questions/13569478</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#2</File><Line>1</Line><Method>module</Method></Frame><Frame><File>D:\my_stuff\Google Drive\Modules\factors.py</File><Line>50</Line><Method>pFactors</Method></Frame><Frame><File>pyshell#3</File><Line>1</Line><Method>module</Method></Frame><Frame><File>D:\my_stuff\Google Drive\Modules\factors.py</File><Line>50</Line><Method>pFactors</Method></Frame></Stack></Post><Post><PostId>13673471</PostId><Url>http://stackoverflow.com/questions/13673471</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/unutbu/pybin/test2.py</File><Line>15</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13692751</PostId><Url>http://stackoverflow.com/questions/13692751</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>test.py</File><Line>19</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13737109</PostId><Url>http://stackoverflow.com/questions/13737109</Url><Stack><Type>IndexError</Type><Message>too many indices</Message><Frame><File>tdm2svd.py</File><Line>40</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13774368</PostId><Url>http://stackoverflow.com/questions/13774368</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#5</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:/Users/Carl/Desktop/try.py</File><Line>53</Line><Method>update</Method></Frame></Stack></Post><Post><PostId>13777469</PostId><Url>http://stackoverflow.com/questions/13777469</Url><Stack><Type>IndexError</Type><Message>pop index out of range</Message><Frame><File>pyshell#54</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/home/wanhrust/python/TSP.py</File><Line>42</Line><Method>min_path</Method></Frame><Frame><File>/home/wanhrust/python/TSP.py</File><Line>42</Line><Method>min_path</Method></Frame><Frame><File>/home/wanhrust/python/TSP.py</File><Line>42</Line><Method>min_path</Method></Frame><Frame><File>/home/wanhrust/python/TSP.py</File><Line>41</Line><Method>min_path</Method></Frame></Stack></Post><Post><PostId>13797829</PostId><Url>http://stackoverflow.com/questions/13797829</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\programs new\codes.py</File><Line>10</Line><Method>module</Method></Frame><Frame><File>C:\Python27\programs new\codes.py</File><Line>10</Line><Method>genexpr</Method></Frame></Stack></Post><Post><PostId>13848506</PostId><Url>http://stackoverflow.com/questions/13848506</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>example.py</File><Line>15</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13850527</PostId><Url>http://stackoverflow.com/questions/13850527</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:/Users/tlindstrom/Desktop/euler.py</File><Line>35</Line><Method>del fiblist[i]</Method></Frame></Stack></Post><Post><PostId>13949716</PostId><Url>http://stackoverflow.com/questions/13949716</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/apps/beat.py</File><Line>100</Line><Method>start_scheduler</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/beat.py</File><Line>422</Line><Method>start</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/beat.py</File><Line>194</Line><Method>tick</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/beat.py</File><Line>172</Line><Method>maybe_due</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/djcelery/schedulers.py</File><Line>65</Line><Method>is_due</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/schedules.py</File><Line>502</Line><Method>is_due</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/schedules.py</File><Line>489</Line><Method>remaining_estimate</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/schedules.py</File><Line>389</Line><Method>_delta_to_next</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/schedules.py</File><Line>372</Line><Method>roll_over</Method></Frame></Stack></Post><Post><PostId>14004880</PostId><Url>http://stackoverflow.com/questions/14004880</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>D:\my_stuff\Google Drive\documents\SCHOOL\Programming\Python\DNA to RNA Converter.py</File><Line>83</Line><Method>module</Method></Frame><Frame><File>D:\my_stuff\Google Drive\documents\SCHOOL\Programming\Python\DNA to RNA Converter.py</File><Line>3</Line><Method>main</Method></Frame><Frame><File>D:\my_stuff\Google Drive\documents\SCHOOL\Programming\Python\DNA to RNA Converter.py</File><Line>58</Line><Method>convert</Method></Frame></Stack></Post><Post><PostId>14141852</PostId><Url>http://stackoverflow.com/questions/14141852</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>odata_search.py</File><Line>46</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14174395</PostId><Url>http://stackoverflow.com/questions/14174395</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\Computing\Unit 3\Hangman\hangmanv2.py</File><Line>200</Line><Method>module</Method></Frame><Frame><File>D:\Computing\Unit 3\Hangman\hangmanv2.py</File><Line>195</Line><Method>hangmangame</Method></Frame><Frame><File>D:\Computing\Unit 3\Hangman\hangmanv2.py</File><Line>187</Line><Method>newwords</Method></Frame><Frame><File>D:\Computing\Unit 3\Hangman\hangmanv2.py</File><Line>158</Line><Method>hangmangame</Method></Frame><Frame><File>D:\Computing\Unit 3\Hangman\hangmanv2.py</File><Line>136</Line><Method>guessletter</Method></Frame></Stack></Post><Post><PostId>14253805</PostId><Url>http://stackoverflow.com/questions/14253805</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>\\3dd15\share\getUNC.py</File><Line>24</Line><Method>GetUnc</Method></Frame><Frame><File>\\3dd15\share\getUNC.py</File><Line>46</Line><Method>GetUncSourcePath</Method></Frame></Stack></Post><Post><PostId>14273124</PostId><Url>http://stackoverflow.com/questions/14273124</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#2</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14400074</PostId><Url>http://stackoverflow.com/questions/14400074</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>bin/funnelweb</File><Line>116</Line><Method>module</Method></Frame><Frame><File>/home/magiq/Plone/buildout-cache/eggs/mr.migrator-1.0.1-py2.7.egg/mr/migrator/runner/__init__.py</File><Line>132</Line><Method>runner</Method></Frame><Frame><File>/home/magiq/Plone/buildout-cache/eggs/collective.transmogrifier-1.3-py2.7.egg/collective/transmogrifier/transmogrifier.py</File><Line>59</Line><Method>__call__</Method></Frame><Frame><File>/home/magiq/Plone/buildout-cache/eggs/collective.transmogrifier-1.3-py2.7.egg/collective/transmogrifier/utils.py</File><Line>52</Line><Method>constructPipeline</Method></Frame><Frame><File>/home/magiq/Plone/buildout-cache/eggs/transmogrify.webcrawler-1.2.1-py2.7.egg/transmogrify/webcrawler/webcrawler.py</File><Line>169</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>14413287</PostId><Url>http://stackoverflow.com/questions/14413287</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./ScrapeTemplate.py</File><Line>29</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14417379</PostId><Url>http://stackoverflow.com/questions/14417379</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#22</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14424842</PostId><Url>http://stackoverflow.com/questions/14424842</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#2</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14427769</PostId><Url>http://stackoverflow.com/questions/14427769</Url><Stack><Type>IndexError</Type><Message>Unexpected SeqBase</Message><Frame><File>/usr/lib/python2.6/lib-tk/Tkinter.py</File><Line>1410</Line><Method>__call__</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/backends/backend_tkagg.py</File><Line>236</Line><Method>resize</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/backends/backend_tkagg.py</File><Line>239</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/backends/backend_agg.py</File><Line>401</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/artist.py</File><Line>55</Line><Method>draw_wrapper</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/figure.py</File><Line>884</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/artist.py</File><Line>55</Line><Method>draw_wrapper</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/axes.py</File><Line>1983</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/artist.py</File><Line>55</Line><Method>draw_wrapper</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/collections.py</File><Line>238</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/python2.6/site-packages/matplotlib/backends/backend_agg.py</File><Line>84</Line><Method>draw_path_collection</Method></Frame></Stack></Post><Post><PostId>14452049</PostId><Url>http://stackoverflow.com/questions/14452049</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>analysis.py</File><Line>276</Line><Method>module</Method></Frame><Frame><File>analysis.py</File><Line>218</Line><Method>parseStringToList</Method></Frame><Frame><File>analysis.py</File><Line>63</Line><Method>listSurgery</Method></Frame><Frame><File>analysis.py</File><Line>62</Line><Method>listSurgery</Method></Frame></Stack></Post><Post><PostId>14459320</PostId><Url>http://stackoverflow.com/questions/14459320</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/base/data/home/apps/s~helloworldtak/1.364765672279579252/hello_world.py</File><Line>16</Line><Method>module</Method></Frame><Frame><File>/python27_runtime/python27_lib/versions/third_party/matplotlib-1.1.1/matplotlib/pyplot.py</File><Line>23</Line><Method>module</Method></Frame><Frame><File>/python27_runtime/python27_lib/versions/third_party/matplotlib-1.1.1/matplotlib/figure.py</File><Line>18</Line><Method>module</Method></Frame><Frame><File>/python27_runtime/python27_lib/versions/third_party/matplotlib-1.1.1/matplotlib/axes.py</File><Line>14</Line><Method>module</Method></Frame><Frame><File>/python27_runtime/python27_lib/versions/third_party/matplotlib-1.1.1/matplotlib/axis.py</File><Line>10</Line><Method>module</Method></Frame><Frame><File>/python27_runtime/python27_lib/versions/third_party/matplotlib-1.1.1/matplotlib/font_manager.py</File><Line>1324</Line><Method>module</Method></Frame><Frame><File>/python27_runtime/python27_lib/versions/third_party/matplotlib-1.1.1/matplotlib/font_manager.py</File><Line>1278</Line><Method>_rebuild</Method></Frame><Frame><File>/python27_runtime/python27_lib/versions/third_party/matplotlib-1.1.1/matplotlib/font_manager.py</File><Line>995</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>14509797</PostId><Url>http://stackoverflow.com/questions/14509797</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>scraper.py</File><Line>48</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14518519</PostId><Url>http://stackoverflow.com/questions/14518519</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#26</File><Line>1</Line><Method>module</Method></Frame><Frame><File>pyshell#25</File><Line>5</Line><Method>match_ends</Method></Frame></Stack></Post><Post><PostId>14531114</PostId><Url>http://stackoverflow.com/questions/14531114</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#121</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14627567</PostId><Url>http://stackoverflow.com/questions/14627567</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#4</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14670887</PostId><Url>http://stackoverflow.com/questions/14670887</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#4</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14684433</PostId><Url>http://stackoverflow.com/questions/14684433</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/opt/openerp/server/openerp/addons/base/ir/ir_cron.py</File><Line>132</Line><Method>_callback</Method></Frame><Frame><File>/opt/openerp/server/openerp/addons/panda_partner/partner.py</File><Line>57</Line><Method>_start_syncro</Method></Frame></Stack></Post><Post><PostId>14765284</PostId><Url>http://stackoverflow.com/questions/14765284</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:\Users\Arjo\Desktop\hw2a.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14791485</PostId><Url>http://stackoverflow.com/questions/14791485</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:/Users/Joloch/PycharmProjects/Varios/Warshall.py</File><Line>93</Line><Method>module</Method></Frame><Frame><File>C:/Users/Joloch/PycharmProjects/Varios/Warshall.py</File><Line>59</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>14888092</PostId><Url>http://stackoverflow.com/questions/14888092</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\***\relogged.pyw</File><Line>15</Line><Method>module</Method></Frame><Frame><File>C:\Users\***\relogged.pyw</File><Line>9</Line><Method>loop</Method></Frame></Stack></Post><Post><PostId>14982912</PostId><Url>http://stackoverflow.com/questions/14982912</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\yarab.py</File><Line>39</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>14989024</PostId><Url>http://stackoverflow.com/questions/14989024</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\Documents\KomodoProjects\Project Euler\ten.py</File><Line>14</Line><Method>module</Method></Frame><Frame><File>D:\Documents\KomodoProjects\Project Euler\ten.py</File><Line>10</Line><Method>go</Method></Frame></Stack></Post><Post><PostId>14991203</PostId><Url>http://stackoverflow.com/questions/14991203</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/PycharmProjects/RestJira/myrestjira.py</File><Line>81</Line><Method>module</Method></Frame><Frame><File>/home/ve/lib/python2.6/site-packages/jira/client.py</File><Line>369</Line><Method>create_issue</Method></Frame><Frame><File>/home/ve/lib/python2.6/site-packages/jira/exceptions.py</File><Line>29</Line><Method>raise_on_error</Method></Frame></Stack></Post><Post><PostId>14993629</PostId><Url>http://stackoverflow.com/questions/14993629</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>E:\LAPTRINH\Android\adt-bundle-windows\eclipse\plugins\org.python.pydev_2.7.1.2012100913\pysrc\pydevd.py</File><Line>1397</Line><Method>module</Method></Frame><Frame><File>E:\LAPTRINH\Android\adt-bundle-windows\eclipse\plugins\org.python.pydev_2.7.1.2012100913\pysrc\pydevd.py</File><Line>1090</Line><Method>run</Method></Frame><Frame><File>C:\Users\The\Documents\workspace\unrar\test_unrar.py</File><Line>13</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15006855</PostId><Url>http://stackoverflow.com/questions/15006855</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\yarab.py</File><Line>38</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15049132</PostId><Url>http://stackoverflow.com/questions/15049132</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>C:\Users\000\Downloads\pyfoobar\pyfoobar\playing.py</File><Line>8</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15246851</PostId><Url>http://stackoverflow.com/questions/15246851</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>make_seq_res.py</File><Line>10</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15281951</PostId><Url>http://stackoverflow.com/questions/15281951</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Scripts\helloscript.py</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15347918</PostId><Url>http://stackoverflow.com/questions/15347918</Url><Stack><Type>IndexError</Type><Message>image index out of range</Message><Frame><File>C:\Documents and Settings\User\My Documents\school\HUB\coding\first attempt.py</File><Line>43</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\PIL\Image.py</File><Line>950</Line><Method>getpixel</Method></Frame></Stack></Post><Post><PostId>15442244</PostId><Url>http://stackoverflow.com/questions/15442244</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>matrixMult.py</File><Line>106</Line><Method>module</Method></Frame><Frame><File>matrixMult.py</File><Line>77</Line><Method>strassen</Method></Frame><Frame><File>matrixMult.py</File><Line>78</Line><Method>strassen</Method></Frame><Frame><File>matrixMult.py</File><Line>82</Line><Method>strassen</Method></Frame><Frame><File>matrixMult.py</File><Line>62</Line><Method>subMatrix</Method></Frame></Stack></Post><Post><PostId>15448722</PostId><Url>http://stackoverflow.com/questions/15448722</Url><Stack><Type>IndexError</Type><Message>item index out of range</Message><Frame><File>code.py</File><Line>261</Line><Method>module</Method></Frame><Frame><File>Full/Path/To/The/module.py</File><Line>37</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>15462060</PostId><Url>http://stackoverflow.com/questions/15462060</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>C:/Python33/Stuff/Ongoing/palindrome.py</File><Line>48</Line><Method>module</Method></Frame><Frame><File>C:/Python33/Stuff/Ongoing/palindrome.py</File><Line>39</Line><Method>ip</Method></Frame></Stack></Post><Post><PostId>15462521</PostId><Url>http://stackoverflow.com/questions/15462521</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Home\Desktop\test.py</File><Line>32</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15668454</PostId><Url>http://stackoverflow.com/questions/15668454</Url><Stack><Type>IndexError</Type><Message>invalid index</Message><Frame><File>test2.py</File><Line>25</Line><Method>module</Method></Frame><Frame><File>/Library/Frameworks/Python.framework/Versions/7.3/lib/python2.7/site-packages/matplotlib/mlab.py</File><Line>2768</Line><Method>griddata</Method></Frame><Frame><File>/Library/Frameworks/Python.framework/Versions/7.3/lib/python2.7/site-packages/matplotlib/delaunay/triangulate.py</File><Line>90</Line><Method>__init__</Method></Frame><Frame><File>/Library/Frameworks/Python.framework/Versions/7.3/lib/python2.7/site-packages/matplotlib/delaunay/triangulate.py</File><Line>115</Line><Method>_compute_convex_hull</Method></Frame></Stack></Post><Post><PostId>15694680</PostId><Url>http://stackoverflow.com/questions/15694680</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>collector2.py</File><Line>81</Line><Method>dataReceived</Method></Frame><Frame><File>/usr/lib/python2.6/pickle.py</File><Line>1374</Line><Method>loads</Method></Frame><Frame><File>/usr/lib/python2.6/pickle.py</File><Line>858</Line><Method>load</Method></Frame><Frame><File>/usr/lib/python2.6/pickle.py</File><Line>1138</Line><Method>load_pop</Method></Frame></Stack></Post><Post><PostId>15714215</PostId><Url>http://stackoverflow.com/questions/15714215</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\default.default-PC\proyectoseclipse\dnsrat\prueba.py</File><Line>8</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15727820</PostId><Url>http://stackoverflow.com/questions/15727820</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#10</File><Line>4</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15764817</PostId><Url>http://stackoverflow.com/questions/15764817</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/amyth/Projects/test/trips/tests.py</File><Line>8</Line><Method>setUp</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/django/db/models/query.py</File><Line>207</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>15778753</PostId><Url>http://stackoverflow.com/questions/15778753</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>pyshell#14</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\statsmodels\tsa\vector_ar\var_model.py</File><Line>336</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\statsmodels\tsa\base\tsa_model.py</File><Line>40</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\statsmodels\tsa\base\tsa_model.py</File><Line>54</Line><Method>_init_dates</Method></Frame><Frame><File>pyshell#47</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\statsmodels\tsa\vector_ar\var_model.py</File><Line>345</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>15778753</PostId><Url>http://stackoverflow.com/questions/15778753</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>pyshell#14</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\statsmodels\tsa\vector_ar\var_model.py</File><Line>336</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\statsmodels\tsa\base\tsa_model.py</File><Line>40</Line><Method>__init__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\statsmodels\tsa\base\tsa_model.py</File><Line>54</Line><Method>_init_dates</Method></Frame><Frame><File>pyshell#47</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\statsmodels\tsa\vector_ar\var_model.py</File><Line>345</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>15824714</PostId><Url>http://stackoverflow.com/questions/15824714</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>prog.py</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15833059</PostId><Url>http://stackoverflow.com/questions/15833059</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\lib\site-packages\web\application.py</File><Line>236</Line><Method>process</Method></Frame><Frame><File>C:\Python27\lib\site-packages\web\application.py</File><Line>227</Line><Method>handle</Method></Frame><Frame><File>C:\Python27\lib\site-packages\web\application.py</File><Line>409</Line><Method>_delegate</Method></Frame><Frame><File>C:\Python27\lib\site-packages\web\application.py</File><Line>384</Line><Method>handle_class</Method></Frame><Frame><File>C:\xampp\htdocs\webpy\code.py</File><Line>32</Line><Method>GET</Method></Frame><Frame><File>C:\xampp\htdocs\webpy\functions.py</File><Line>16</Line><Method>__init__</Method></Frame><Frame><File>C:\xampp\htdocs\webpy\functions.py</File><Line>26</Line><Method>getCat</Method></Frame></Stack></Post><Post><PostId>15842249</PostId><Url>http://stackoverflow.com/questions/15842249</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#241</File><Line>3</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>15919158</PostId><Url>http://stackoverflow.com/questions/15919158</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>pyshell#34</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>16020597</PostId><Url>http://stackoverflow.com/questions/16020597</Url><Stack><Type>IndexError</Type><Message>Index Out of range for SearchError object</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>14</Line><Method>__getitem__</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>21</Line><Method>__getitem__</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>21</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>16020597</PostId><Url>http://stackoverflow.com/questions/16020597</Url><Stack><Type>IndexError</Type><Message>Index Out of range for SearchError object</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>14</Line><Method>__getitem__</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>21</Line><Method>__getitem__</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>21</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>16020597</PostId><Url>http://stackoverflow.com/questions/16020597</Url><Stack><Type>IndexError</Type><Message>Index Out of range for SearchError object</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>14</Line><Method>__getitem__</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>21</Line><Method>__getitem__</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>21</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>16143077</PostId><Url>http://stackoverflow.com/questions/16143077</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>path</File><Line>66</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>16185995</PostId><Url>http://stackoverflow.com/questions/16185995</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>/usr/lib/python3.1/sre_parse.py</File><Line>194</Line><Method>__next</Method></Frame></Stack></Post><Post><PostId>16257216</PostId><Url>http://stackoverflow.com/questions/16257216</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range"</Message><Frame><File>C:\Python33\HW3 playing around.py</File><Line>133</Line><Method>module</Method></Frame><Frame><File>C:\Python33\HW3 playing around.py</File><Line>32</Line><Method>Main</Method></Frame><Frame><File>C:\Python33\HW3 playing around.py</File><Line>69</Line><Method>EmployeeNumbers</Method></Frame></Stack></Post><Post><PostId>16260265</PostId><Url>http://stackoverflow.com/questions/16260265</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/Users/joelwilliams/Desktop/delete me</File><Line>30</Line><Method>module</Method></Frame><Frame><File>/Users/joelwilliams/Desktop/delete me</File><Line>24</Line><Method>writef</Method></Frame><Frame><File>/Users/joelwilliams/Desktop/delete me</File><Line>15</Line><Method>write</Method></Frame></Stack></Post><Post><PostId>16276536</PostId><Url>http://stackoverflow.com/questions/16276536</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/tmp/hadoop-hduser/mapred/local/taskTracker/hduser/jobcache/job_201304232210_0007/attempt_201304232210_0007_m_000001_3/work/./CountWordOccurence_mapper.py</File><Line>6</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>16285816</PostId><Url>http://stackoverflow.com/questions/16285816</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>genexpr</Method></Frame></Stack></Post><Post><PostId>16380254</PostId><Url>http://stackoverflow.com/questions/16380254</Url><Stack><Type>IndexError</Type><Message>index 3 is out of bounds for axis 0 with size 3</Message><Frame><File>pyshell#27</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>16465106</PostId><Url>http://stackoverflow.com/questions/16465106</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\lib\threading.py</File><Line>551</Line><Method>__bootstrap_inner</Method></Frame><Frame><File>C:\Python27\lib\threading.py</File><Line>504</Line><Method>run</Method></Frame><Frame><File>fakespider.py</File><Line>37</Line><Method>proxy</Method></Frame></Stack></Post><Post><PostId>16545411</PostId><Url>http://stackoverflow.com/questions/16545411</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./test_detection.py</File><Line>6</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>16562707</PostId><Url>http://stackoverflow.com/questions/16562707</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>C:\Python27\lib\lib-tk\Tkinter.py</File><Line>1410</Line><Method>__call__</Method></Frame><Frame><File>D:/Google Drive/assign2_2-1.py</File><Line>113</Line><Method>add_func</Method></Frame><Frame><File>D:/Google Drive/assign2_2-1.py</File><Line>132</Line><Method>redraw_all</Method></Frame><Frame><File>D:/Google Drive/assign2_2-1.py</File><Line>145</Line><Method>draw</Method></Frame><Frame><File>C:\Python27\lib\lib-tk\Tkinter.py</File><Line>2201</Line><Method>create_line</Method></Frame><Frame><File>C:\Python27\lib\lib-tk\Tkinter.py</File><Line>2182</Line><Method>_create</Method></Frame></Stack></Post><Post><PostId>16660003</PostId><Url>http://stackoverflow.com/questions/16660003</Url><Stack><Type>IndexError</Type><Message>Cannot choose from an empty sequence</Message><Frame><File>/usr/lib/python3.3/random.py</File><Line>249</Line><Method>choice</Method></Frame><Frame><File>/usr/lib/python3.3/random.py</File><Line>225</Line><Method>_randbelow</Method></Frame><Frame><File>game_code.py</File><Line>58</Line><Method>button_clicked</Method></Frame><Frame><File>/usr/lib/python3.3/random.py</File><Line>251</Line><Method>choice</Method></Frame></Stack></Post><Post><PostId>16660003</PostId><Url>http://stackoverflow.com/questions/16660003</Url><Stack><Type>IndexError</Type><Message>Cannot choose from an empty sequence</Message><Frame><File>/usr/lib/python3.3/random.py</File><Line>249</Line><Method>choice</Method></Frame><Frame><File>/usr/lib/python3.3/random.py</File><Line>225</Line><Method>_randbelow</Method></Frame><Frame><File>game_code.py</File><Line>58</Line><Method>button_clicked</Method></Frame><Frame><File>/usr/lib/python3.3/random.py</File><Line>251</Line><Method>choice</Method></Frame></Stack></Post><Post><PostId>16662411</PostId><Url>http://stackoverflow.com/questions/16662411</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range*</Message><Frame><File>./bio.py</File><Line>25</Line><Method>module</Method></Frame><Frame><File>./bio.py</File><Line>9</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>16745487</PostId><Url>http://stackoverflow.com/questions/16745487</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/mcapp/.virtualenv/lister/local/lib/python2.7/site-packages/celery/task/trace.py</File><Line>228</Line><Method>trace_task</Method></Frame><Frame><File>/home/mcapp/.virtualenv/lister/local/lib/python2.7/site-packages/celery/task/trace.py</File><Line>415</Line><Method>strong protected_call /strong</Method></Frame><Frame><File>/home/mcapp/lister/lister/tasks/ strong init /strong .py</File><Line>69</Line><Method>update_playlist_db</Method></Frame><Frame><File>/home/mcapp/lister/lister/tasks/ strong init /strong .py</File><Line>55</Line><Method>video_update</Method></Frame><Frame><File>/home/mcapp/.virtualenv/lister/local/lib/python2.7/site-packages/celery/canvas.py</File><Line>464</Line><Method>strong call /strong</Method></Frame><Frame><File>/home/mcapp/.virtualenv/lister/local/lib/python2.7/site-packages/celery/canvas.py</File><Line>461</Line><Method>type</Method></Frame></Stack></Post><Post><PostId>16765819</PostId><Url>http://stackoverflow.com/questions/16765819</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>/usr/lib/python2.7/lib-tk/Tkinter.py</File><Line>1473</Line><Method>__call__</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py</File><Line>276</Line><Method>resize</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py</File><Line>348</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/backends/backend_agg.py</File><Line>440</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/artist.py</File><Line>54</Line><Method>draw_wrapper</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/figure.py</File><Line>1006</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/artist.py</File><Line>54</Line><Method>draw_wrapper</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/axes.py</File><Line>2086</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/artist.py</File><Line>54</Line><Method>draw_wrapper</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/axis.py</File><Line>1052</Line><Method>draw</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/axis.py</File><Line>1001</Line><Method>_get_tick_bboxes</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/text.py</File><Line>752</Line><Method>get_window_extent</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/text.py</File><Line>313</Line><Method>_get_layout</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/backends/backend_agg.py</File><Line>200</Line><Method>get_text_width_height_descent</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/texmanager.py</File><Line>606</Line><Method>get_text_width_height_descent</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/dviread.py</File><Line>71</Line><Method>__iter__</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/dviread.py</File><Line>126</Line><Method>_read</Method></Frame></Stack></Post><Post><PostId>16882612</PostId><Url>http://stackoverflow.com/questions/16882612</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>uv.py</File><Line>4</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>16952655</PostId><Url>http://stackoverflow.com/questions/16952655</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>lambda</Method></Frame></Stack></Post><Post><PostId>17000090</PostId><Url>http://stackoverflow.com/questions/17000090</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>temp.py</File><Line>18</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>17037818</PostId><Url>http://stackoverflow.com/questions/17037818</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>hello.py</File><Line>35</Line><Method>module</Method></Frame><Frame><File>hello.py</File><Line>31</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>17095958</PostId><Url>http://stackoverflow.com/questions/17095958</Url><Stack><Type>IndexError</Type><Message>arrays used as indices must be of integer (or boolean) type</Message><Frame><File>lpi2.py</File><Line>12</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>17119130</PostId><Url>http://stackoverflow.com/questions/17119130</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./count_aa.py</File><Line>4</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>17130709</PostId><Url>http://stackoverflow.com/questions/17130709</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>prittyprint.py</File><Line>17</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>17269149</PostId><Url>http://stackoverflow.com/questions/17269149</Url><Stack><Type>IndexError</Type><Message>invalid index to scalar variable.</Message><Frame><File>pyshell#362</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:\Python31\lib\site-packages\scipy\optimize\optimize.py</File><Line>258</Line><Method>fmin</Method></Frame><Frame><File>C:\Python31\lib\site-packages\scipy\optimize\optimize.py</File><Line>177</Line><Method>function_wrapper</Method></Frame><Frame><File>pyshell#361</File><Line>7</Line><Method>residue</Method></Frame></Stack></Post><Post><PostId>17285691</PostId><Url>http://stackoverflow.com/questions/17285691</Url><Stack><Type>IndexError</Type><Message>too many indices for array</Message><Frame><File>pyshell#38</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>17318545</PostId><Url>http://stackoverflow.com/questions/17318545</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/borko/Desktop/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>1535</Line><Method>__call__</Method></Frame><Frame><File>/home/borko/Desktop/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>1529</Line><Method>__call__</Method></Frame><Frame><File>/home/borko/Desktop/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>1278</Line><Method>default_dispatcher</Method></Frame><Frame><File>/home/borko/Desktop/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>1102</Line><Method>__call__</Method></Frame><Frame><File>/home/borko/Desktop/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>572</Line><Method>dispatch</Method></Frame><Frame><File>/home/borko/Desktop/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>570</Line><Method>dispatch</Method></Frame><Frame><File>/home/borko/gae/submissions/handlers.py</File><Line>26</Line><Method>post</Method></Frame></Stack></Post><Post><PostId>17339107</PostId><Url>http://stackoverflow.com/questions/17339107</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/michael/std/dev/python/DM/DM0.0.2.py</File><Line>56</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>17385570</PostId><Url>http://stackoverflow.com/questions/17385570</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Usman\Documents\Developer\Python\Pearson Correlation\pearson_ce.py</File><Line>34</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>17393989</PostId><Url>http://stackoverflow.com/questions/17393989</Url><Stack><Type>IndexError</Type><Message>arrays used as indices must be of integer (or boolean) type</Message><Frame><File>NearestCentroid.py</File><Line>53</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/scikit_learn-0.13.1-py2.7-linux-i686.egg/sklearn/neighbors/nearest_centroid.py</File><Line>115</Line><Method>fit</Method></Frame></Stack></Post><Post><PostId>17414098</PostId><Url>http://stackoverflow.com/questions/17414098</Url><Stack><Type>IndexError</Type><Message>0-d arrays can't be indexed.</Message><Frame><File>ipython-input-50-91c06aa4487b</File><Line>1</Line><Method>module</Method></Frame><Frame><File>script.py</File><Line>22</Line><Method>step_mask</Method></Frame><Frame><File>ipython-input-51-4e0de4fcb197</File><Line>1</Line><Method>module</Method></Frame><Frame><File>script.py</File><Line>55</Line><Method>step_for</Method></Frame></Stack></Post><Post><PostId>17414098</PostId><Url>http://stackoverflow.com/questions/17414098</Url><Stack><Type>IndexError</Type><Message>0-d arrays can't be indexed.</Message><Frame><File>ipython-input-50-91c06aa4487b</File><Line>1</Line><Method>module</Method></Frame><Frame><File>script.py</File><Line>22</Line><Method>step_mask</Method></Frame><Frame><File>ipython-input-51-4e0de4fcb197</File><Line>1</Line><Method>module</Method></Frame><Frame><File>script.py</File><Line>55</Line><Method>step_for</Method></Frame></Stack></Post><Post><PostId>17460779</PostId><Url>http://stackoverflow.com/questions/17460779</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\nasa\nasa_god.py</File><Line>11</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>17464918</PostId><Url>http://stackoverflow.com/questions/17464918</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>insertion.py</File><Line>6</Line><Method>test_sort</Method></Frame><Frame><File>insertion.py</File><Line>12</Line><Method>sort</Method></Frame></Stack></Post><Post><PostId>17486298</PostId><Url>http://stackoverflow.com/questions/17486298</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>annotate.py</File><Line>229</Line><Method>module</Method></Frame><Frame><File>random.pyc</File><Line>274</Line><Method>choice</Method></Frame></Stack></Post><Post><PostId>17519323</PostId><Url>http://stackoverflow.com/questions/17519323</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#1</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:/Users/Manmohit/Desktop/anonymiser.py</File><Line>11</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>17551671</PostId><Url>http://stackoverflow.com/questions/17551671</Url><Stack><Type>IndexError</Type><Message>invalid slice</Message><Frame><File>/Users/geoffroysarrazin/Desktop/stage_observatoire/testEkki.py</File><Line>52</Line><Method>essai</Method></Frame></Stack></Post><Post><PostId>17579600</PostId><Url>http://stackoverflow.com/questions/17579600</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Lolpin\Desktop\fishinglooptest.py</File><Line>308</Line><Method>module</Method></Frame><Frame><File>C:\Users\Lolpin\Desktop\fishinglooptest.py</File><Line>231</Line><Method>action_function</Method></Frame><Frame><File>C:\Users\Lolpin\Desktop\fishinglooptest.py</File><Line>40</Line><Method>sell_function</Method></Frame></Stack></Post><Post><PostId>17646626</PostId><Url>http://stackoverflow.com/questions/17646626</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/Users/Kyle/Library/Application Support/Titanium/mobilesdk/osx/2.0.1.GA2/iphone/builder.py</File><Line>1334</Line><Method>main</Method></Frame><Frame><File>/Users/Kyle/Library/Application Support/Titanium/mobilesdk/osx/2.0.1.GA2/iphone/builder.py</File><Line>508</Line><Method>cleanup_app_logfiles</Method></Frame><Frame><File>/Users/Kyle/Library/Application Support/Titanium/mobilesdk/osx/2.0.1.GA2/iphone/builder.py</File><Line>411</Line><Method>is_indexing_enabled</Method></Frame></Stack></Post><Post><PostId>17718378</PostId><Url>http://stackoverflow.com/questions/17718378</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Vaibhav\Desktop\Webtrends\test.py</File><Line>43</Line><Method>xmlTag = dom.getElementsByTagName('string name="ID"')[0].toxml()</Method></Frame></Stack></Post><Post><PostId>17773480</PostId><Url>http://stackoverflow.com/questions/17773480</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#75</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18110513</PostId><Url>http://stackoverflow.com/questions/18110513</Url><Stack><Type>IndexError</Type><Message>index 1 is out of bounds for axis 0 with size 1</Message><Frame><File>tsp_solver.py</File><Line>57</Line><Method>module</Method></Frame><Frame><File>tsp_solver.py</File><Line>57</Line><Method>module</Method></Frame><Frame><File>tsp_solver.py</File><Line>36</Line><Method>solveTSP</Method></Frame><Frame><File>/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-    packages/openopt-0.506-py2.7.egg/openopt/kernel/TSP.py</File><Line>150</Line><Method>solve</Method></Frame></Stack></Post><Post><PostId>18110513</PostId><Url>http://stackoverflow.com/questions/18110513</Url><Stack><Type>IndexError</Type><Message>index 1 is out of bounds for axis 0 with size 1</Message><Frame><File>tsp_solver.py</File><Line>57</Line><Method>module</Method></Frame><Frame><File>tsp_solver.py</File><Line>57</Line><Method>module</Method></Frame><Frame><File>tsp_solver.py</File><Line>36</Line><Method>solveTSP</Method></Frame><Frame><File>/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-    packages/openopt-0.506-py2.7.egg/openopt/kernel/TSP.py</File><Line>150</Line><Method>solve</Method></Frame></Stack></Post><Post><PostId>18112886</PostId><Url>http://stackoverflow.com/questions/18112886</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Ben\Documents\Python Files\PlatformerGame</File><Line>89</Line><Method>module</Method></Frame><Frame><File>C:\Users\Ben\Documents\Python Files\PlatformerGame</File><Line>42</Line><Method>__main__</Method></Frame></Stack></Post><Post><PostId>18198824</PostId><Url>http://stackoverflow.com/questions/18198824</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\lib\site-packages\IPython\core\ultratb.py</File><Line>779</Line><Method>structured_traceback</Method></Frame><Frame><File>C:\Python27\lib\site-packages\IPython\core\ultratb.py</File><Line>245</Line><Method>_fixed_getinnerframes</Method></Frame><Frame><File>C:\Python27\lib\inspect.py</File><Line>1043</Line><Method>getinnerframes</Method></Frame><Frame><File>C:\Python27\lib\inspect.py</File><Line>1007</Line><Method>getframeinfo</Method></Frame><Frame><File>C:\Python27\lib\inspect.py</File><Line>580</Line><Method>findsource</Method></Frame></Stack></Post><Post><PostId>18259094</PostId><Url>http://stackoverflow.com/questions/18259094</Url><Stack><Type>IndexError</Type><Message>abc</Message><Frame><File>pyshell#22</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18266615</PostId><Url>http://stackoverflow.com/questions/18266615</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>131</Line><Method>wrapper</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>197</Line><Method>execute</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>185</Line><Method>execute_cr</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/addons/hr_attendance/hr_attendance.py</File><Line>227</Line><Method>text_files_upload</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>131</Line><Method>wrapper</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>197</Line><Method>execute</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>185</Line><Method>execute_cr</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/addons/hr_attendance/hr_attendance.py</File><Line>228</Line><Method>text_files_upload</Method></Frame></Stack></Post><Post><PostId>18266615</PostId><Url>http://stackoverflow.com/questions/18266615</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>131</Line><Method>wrapper</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>197</Line><Method>execute</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>185</Line><Method>execute_cr</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/addons/hr_attendance/hr_attendance.py</File><Line>227</Line><Method>text_files_upload</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>131</Line><Method>wrapper</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>197</Line><Method>execute</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/osv/osv.py</File><Line>185</Line><Method>execute_cr</Method></Frame><Frame><File>/home/priyan/hr_openerp/openerp-7.0/openerp/addons/hr_attendance/hr_attendance.py</File><Line>228</Line><Method>text_files_upload</Method></Frame></Stack></Post><Post><PostId>18452512</PostId><Url>http://stackoverflow.com/questions/18452512</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>terima.py</File><Line>11</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18473161</PostId><Url>http://stackoverflow.com/questions/18473161</Url><Stack><Type>IndexError</Type><Message>0-d arrays can only use a single () or a list of newaxes (and a single ...) as an index</Message><Frame><File>/opt/local/Library/Frameworks/Python.framework/Versions/3.3/lib/python3.3/site-packages/IPython/core/interactiveshell.py</File><Line>2732</Line><Method>run_code</Method></Frame><Frame><File>ipython-input-1-35f1349fb755</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/opt/local/Library/Frameworks/Python.framework/Versions/3.3/lib/python3.3/site-packages/IPython/core/interactiveshell.py</File><Line>2732</Line><Method>run_code</Method></Frame><Frame><File>ipython-input-1-43c5da404060</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18473161</PostId><Url>http://stackoverflow.com/questions/18473161</Url><Stack><Type>IndexError</Type><Message>0-d arrays can only use a single () or a list of newaxes (and a single ...) as an index</Message><Frame><File>/opt/local/Library/Frameworks/Python.framework/Versions/3.3/lib/python3.3/site-packages/IPython/core/interactiveshell.py</File><Line>2732</Line><Method>run_code</Method></Frame><Frame><File>ipython-input-1-35f1349fb755</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/opt/local/Library/Frameworks/Python.framework/Versions/3.3/lib/python3.3/site-packages/IPython/core/interactiveshell.py</File><Line>2732</Line><Method>run_code</Method></Frame><Frame><File>ipython-input-1-43c5da404060</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18588858</PostId><Url>http://stackoverflow.com/questions/18588858</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>program.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18633457</PostId><Url>http://stackoverflow.com/questions/18633457</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Users/lapje/Documents/Naam.py</File><Line>5</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18634136</PostId><Url>http://stackoverflow.com/questions/18634136</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Michele2\Desktop\Arcrate\MyCASH\Python Code\Scraping\Scraping1.3(clean)TEST.py</File><Line>246</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18654219</PostId><Url>http://stackoverflow.com/questions/18654219</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>Lab1.py</File><Line>24</Line><Method>module</Method></Frame><Frame><File>Lab1.py</File><Line>19</Line><Method>mergesort</Method></Frame><Frame><File>Lab1.py</File><Line>21</Line><Method>mergesort</Method></Frame><Frame><File>Lab1.py</File><Line>12</Line><Method>merge</Method></Frame></Stack></Post><Post><PostId>18684663</PostId><Url>http://stackoverflow.com/questions/18684663</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>C:/Documents and Settings/All Users/Documents/python/caeser hacker.py</File><Line>27</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>18721855</PostId><Url>http://stackoverflow.com/questions/18721855</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>external_linaro_toolchain_version_handler(e)</File><Line>8</Line><Method>external_linaro_toolchain_version_handler(e= bb.event.ConfigParsed object at 0x17ed4d0 )</Method></Frame><Frame><File>external-linaro-toolchain-versions.inc</File><Line>4</Line><Method>elt_get_main_version(d= bb.data_smart.DataSmart object at 0x17ed390 )</Method></Frame></Stack></Post><Post><PostId>18858396</PostId><Url>http://stackoverflow.com/questions/18858396</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>pyshell#76</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19009053</PostId><Url>http://stackoverflow.com/questions/19009053</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#66</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/matplotlib    /pyplot.py</File><Line>2442</Line><Method>boxplot</Method></Frame><Frame><File>/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/matplotlib/axes.py</File><Line>5815</Line><Method>boxplot</Method></Frame></Stack></Post><Post><PostId>19020348</PostId><Url>http://stackoverflow.com/questions/19020348</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#135</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19028084</PostId><Url>http://stackoverflow.com/questions/19028084</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/data/hadoop-admin/mapred/local/taskTracker/admin/jobcache/job_201309130717_0378/attempt_201309130717_0378_r_000000_0/work/./SessionJsonReducer.py</File><Line>98</Line><Method>module</Method></Frame><Frame><File>/data/hadoop-admin/mapred/local/taskTracker/admin/jobcache/job_201309130717_0378/attempt_201309130717_0378_r_000000_0/work/./SessionJsonReducer.py</File><Line>41</Line><Method>createMap</Method></Frame></Stack></Post><Post><PostId>19068586</PostId><Url>http://stackoverflow.com/questions/19068586</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>vm_main.py</File><Line>33</Line><Method>module</Method></Frame><Frame><File>/tmp/vmuser_trwqmlfqgq/main.py</File><Line>143</Line><Method>module</Method></Frame><Frame><File>/tmp/vmuser_trwqmlfqgq/main.py</File><Line>95</Line><Method>cellular_automaton</Method></Frame></Stack></Post><Post><PostId>19076007</PostId><Url>http://stackoverflow.com/questions/19076007</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/misha/Documents/finance/finance.py</File><Line>164</Line><Method>module</Method></Frame><Frame><File>/home/misha/Documents/finance/finance.py</File><Line>125</Line><Method>security_mean</Method></Frame><Frame><File>/usr/lib/python2.7/site-packages/numpy/lib/npyio.py</File><Line>825</Line><Method>loadtxt</Method></Frame></Stack></Post><Post><PostId>19173937</PostId><Url>http://stackoverflow.com/questions/19173937</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>./demo.py</File><Line>37</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19259487</PostId><Url>http://stackoverflow.com/questions/19259487</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\NAME\Desktop\storeLocation.py</File><Line>32</Line><Method>module</Method></Frame><Frame><File>C:\Users\NAME\Desktop\storeLocation.py</File><Line>30</Line><Method>main</Method></Frame><Frame><File>C:\Users\NAME\Desktop\storeLocation.py</File><Line>14</Line><Method>getBestLoc</Method></Frame></Stack></Post><Post><PostId>19262268</PostId><Url>http://stackoverflow.com/questions/19262268</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./add.py</File><Line>3</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19289035</PostId><Url>http://stackoverflow.com/questions/19289035</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>SearchString.py</File><Line>46</Line><Method>module</Method></Frame><Frame><File>SearchString.py</File><Line>36</Line><Method>IndexSearchString</Method></Frame></Stack></Post><Post><PostId>19298147</PostId><Url>http://stackoverflow.com/questions/19298147</Url><Stack><Type>IndexError</Type><Message>list index out of range)</Message><Frame><File>E:/ankosh/trial13.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19338268</PostId><Url>http://stackoverflow.com/questions/19338268</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>/Users/ranbir/Desktop/names/top101.py</File><Line>20</Line><Method>module</Method></Frame><Frame><File>/Users/ranbir/Desktop/names/top101.py</File><Line>18</Line><Method>main</Method></Frame><Frame><File>/Users/ranbir/Desktop/names/top101.py</File><Line>12</Line><Method>mkEntry</Method></Frame></Stack></Post><Post><PostId>19339825</PostId><Url>http://stackoverflow.com/questions/19339825</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\RawDataeDataTest.py</File><Line>7</Line><Method>module</Method></Frame><Frame><File>C:\RawDataeDataTest.py</File><Line>7</Line><Method>lambda</Method></Frame></Stack></Post><Post><PostId>19365356</PostId><Url>http://stackoverflow.com/questions/19365356</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\python scriptname.py</File><Line>9</Line><Method>host = sys.argv[1]</Method></Frame></Stack></Post><Post><PostId>19367389</PostId><Url>http://stackoverflow.com/questions/19367389</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#7</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19455426</PostId><Url>http://stackoverflow.com/questions/19455426</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>C:\Users\Killian\workspace\College\Assignment.py</File><Line>25</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19486706</PostId><Url>http://stackoverflow.com/questions/19486706</Url><Stack><Type>IndexError</Type><Message>no such group</Message><Frame><File>C:\Users\*******\workspace\Jython\regularexpressions.py</File><Line>35</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19522991</PostId><Url>http://stackoverflow.com/questions/19522991</Url><Stack><Type>IndexError</Type><Message>index 1 is out of bounds for axis 0 with size 1</Message><Frame><File>pyshell#89</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19578975</PostId><Url>http://stackoverflow.com/questions/19578975</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#3</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:/Users/***/Documents/prog123</File><Line>28</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>19588465</PostId><Url>http://stackoverflow.com/questions/19588465</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>a.py</File><Line>54</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19596499</PostId><Url>http://stackoverflow.com/questions/19596499</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>6</Line><Method>guess</Method></Frame></Stack></Post><Post><PostId>19604825</PostId><Url>http://stackoverflow.com/questions/19604825</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>E:/python/alphabetical.py</File><Line>13</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19628066</PostId><Url>http://stackoverflow.com/questions/19628066</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#54</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/Users/user/Desktop/Outlook(1)/lab6.py</File><Line>59</Line><Method>test</Method></Frame><Frame><File>pyshell#56</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/Users/user/Desktop/Outlook(1)/lab6.py</File><Line>58</Line><Method>test</Method></Frame></Stack></Post><Post><PostId>19628066</PostId><Url>http://stackoverflow.com/questions/19628066</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#54</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/Users/user/Desktop/Outlook(1)/lab6.py</File><Line>59</Line><Method>test</Method></Frame><Frame><File>pyshell#56</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/Users/user/Desktop/Outlook(1)/lab6.py</File><Line>58</Line><Method>test</Method></Frame></Stack></Post><Post><PostId>19631798</PostId><Url>http://stackoverflow.com/questions/19631798</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>I:\Workspace\openobject-6.0\openobject-server\6.0\bin\netsvc.py</File><Line>489</Line><Method>dispatch</Method></Frame><Frame><File>I:\Workspace\openobject-6.0\openobject-server\6.0\bin\service\web_services.py</File><Line>599</Line><Method>dispatch</Method></Frame><Frame><File>I:\Workspace\openobject-6.0\openobject-server\6.0\bin\osv\osv.py</File><Line>129</Line><Method>wrapper</Method></Frame><Frame><File>I:\Workspace\openobject-6.0\openobject-server\6.0\bin\osv\osv.py</File><Line>194</Line><Method>execute</Method></Frame><Frame><File>I:\Workspace\openobject-6.0\openobject-server\6.0\bin\osv\osv.py</File><Line>185</Line><Method>execute_cr</Method></Frame><Frame><File>I:\Workspace\openobject-6.0\openobject-server\6.0\bin\osv\orm.py</File><Line>1650</Line><Method>fields_view_get</Method></Frame><Frame><File>I:\Workspace\openobject-6.0\openobject-server\6.0\bin\osv\orm.py</File><Line>1340</Line><Method>__view_look_dom_arch</Method></Frame></Stack></Post><Post><PostId>19650528</PostId><Url>http://stackoverflow.com/questions/19650528</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>test.py</File><Line>102</Line><Method>module</Method></Frame><Frame><File>test.py</File><Line>66</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>19678031</PostId><Url>http://stackoverflow.com/questions/19678031</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>1535</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>1529</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>1278</Line><Method>default_dispatcher</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>1102</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>572</Line><Method>dispatch</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>570</Line><Method>dispatch</Method></Frame><Frame><File>/base/data/home/apps/s~map-jobs/2.371278632227250958/main.py</File><Line>453</Line><Method>post</Method></Frame><Frame><File>/base/data/home/apps/s~map-jobs/2.371278632227250958/main.py</File><Line>17</Line><Method>get_lat_long</Method></Frame></Stack></Post><Post><PostId>19837928</PostId><Url>http://stackoverflow.com/questions/19837928</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>pyshell#75</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19849635</PostId><Url>http://stackoverflow.com/questions/19849635</Url><Stack><Type>IndexError</Type><Message>too many indices</Message><Frame><File>z-matrix-filtering.py</File><Line>11</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>19852461</PostId><Url>http://stackoverflow.com/questions/19852461</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\preprocessing adding adnan.py</File><Line>51</Line><Method>lines.append(lines[i+1])</Method></Frame></Stack></Post><Post><PostId>19883776</PostId><Url>http://stackoverflow.com/questions/19883776</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>getArticleInformation.py</File><Line>69</Line><Method>module</Method></Frame><Frame><File>getArticleInformation.py</File><Line>63</Line><Method>printCSV</Method></Frame></Stack></Post><Post><PostId>19928051</PostId><Url>http://stackoverflow.com/questions/19928051</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>manage.py</File><Line>10</Line><Method>module</Method></Frame><Frame><File>/home/vagrant/.virtualenvs/rsspainter/local/lib/python2.7/site-packages/django/core/management/__init__.py</File><Line>453</Line><Method>execute_from_command_line</Method></Frame><Frame><File>/home/vagrant/.virtualenvs/rsspainter/local/lib/python2.7/site-packages/django/core/management/__init__.py</File><Line>392</Line><Method>execute</Method></Frame><Frame><File>/home/vagrant/.virtualenvs/rsspainter/local/lib/python2.7/site-packages/django/core/management/__init__.py</File><Line>272</Line><Method>fetch_command</Method></Frame><Frame><File>/home/vagrant/.virtualenvs/rsspainter/local/lib/python2.7/site-packages/django/core/management/__init__.py</File><Line>77</Line><Method>load_command_class</Method></Frame><Frame><File>/home/vagrant/.virtualenvs/rsspainter/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/vagrant/rsspainter/apps/account/management/commands/create_users.py</File><Line>5</Line><Method>module</Method></Frame><Frame><File>/vagrant/rsspainter/apps/account/models.py</File><Line>5</Line><Method>module</Method></Frame><Frame><File>/home/vagrant/.virtualenvs/rsspainter/local/lib/python2.7/site-packages/django/db/models/base.py</File><Line>93</Line><Method>__new__</Method></Frame></Stack></Post><Post><PostId>20026343</PostId><Url>http://stackoverflow.com/questions/20026343</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./rst2html.py</File><Line>24</Line><Method>module</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/core.py</File><Line>352</Line><Method>publish_cmdline</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/core.py</File><Line>219</Line><Method>publish</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/writers/__init__.py</File><Line>80</Line><Method>write</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/writers/html4css1/__init__.py</File><Line>173</Line><Method>translate</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/nodes.py</File><Line>174</Line><Method>walkabout</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/nodes.py</File><Line>174</Line><Method>walkabout</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/nodes.py</File><Line>174</Line><Method>walkabout</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/nodes.py</File><Line>174</Line><Method>walkabout</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/nodes.py</File><Line>174</Line><Method>walkabout</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/nodes.py</File><Line>174</Line><Method>walkabout</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/nodes.py</File><Line>166</Line><Method>walkabout</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/nodes.py</File><Line>1627</Line><Method>dispatch_visit</Method></Frame><Frame><File>/usr/lib64/python3.2/site-packages/docutils/writers/html4css1/__init__.py</File><Line>792</Line><Method>visit_entry</Method></Frame></Stack></Post><Post><PostId>20084356</PostId><Url>http://stackoverflow.com/questions/20084356</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:/Sheridan/Part Time/TELE26529 Linux Architecture w. Network Scripting/Python Assignment 3.5/question1.py</File><Line>17</Line><Method>module</Method></Frame><Frame><File>D:/Sheridan/Part Time/TELE26529 Linux Architecture w. Network Scripting/Python Assignment 3.5/question1.py</File><Line>14</Line><Method>grab_email</Method></Frame></Stack></Post><Post><PostId>20149966</PostId><Url>http://stackoverflow.com/questions/20149966</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>file.py</File><Line>21</Line><Method>module</Method></Frame><Frame><File>file.py</File><Line>20</Line><Method>main</Method></Frame><Frame><File>file.py</File><Line>5</Line><Method>small</Method></Frame></Stack></Post><Post><PostId>20212004</PostId><Url>http://stackoverflow.com/questions/20212004</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>LINKTOFILEHERE</File><Line>9</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>20308165</PostId><Url>http://stackoverflow.com/questions/20308165</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Vinicius\Documents\GitHub\python-crawler\scrapper-2-0.py</File><Line>18</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>20308608</PostId><Url>http://stackoverflow.com/questions/20308608</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>test2.py</File><Line>72</Line><Method>module</Method></Frame><Frame><File>test2.py</File><Line>23</Line><Method>url_queue</Method></Frame><Frame><File>test2.py</File><Line>13</Line><Method>convert_to_link</Method></Frame><Frame><File>test2.py</File><Line>62</Line><Method>evaluate</Method></Frame></Stack></Post><Post><PostId>20396956</PostId><Url>http://stackoverflow.com/questions/20396956</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#0</File><Line>1</Line><Method>module  is_palidrome_v3('noon')</Method></Frame><Frame><File>C:\Users\James\Google-h0925473\Learning Programming\Python\Python Fundamental (Coursera)\is_palidrome_v1.py</File><Line>67</Line><Method>is_palidrome_v3 while i  = len(s) // 2 and s[i] == s[len(s) - i]</Method></Frame></Stack></Post><Post><PostId>20432328</PostId><Url>http://stackoverflow.com/questions/20432328</Url><Stack><Type>IndexError</Type><Message>0-d arrays can only use a single () or a list of newaxes (and a single ...) as an index</Message><Frame><File>C:\Users\Ano\workspace\final_submission\src\rf.py</File><Line>38</Line><Method>module</Method></Frame><Frame><File>C:\Users\Ano\workspace\final_submission\src\rf.py</File><Line>18</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>20457864</PostId><Url>http://stackoverflow.com/questions/20457864</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>C:\Documents and Settings\ususer\My Documents\Downloads\StringStuff.py</File><Line>27</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>20462382</PostId><Url>http://stackoverflow.com/questions/20462382</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>parser.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>20473137</PostId><Url>http://stackoverflow.com/questions/20473137</Url><Stack><Type>IndexError</Type><Message>pop from empty list</Message><Frame><File>/usr/local/lib/python2.7/dist-packages/tornado/web.py</File><Line>1115</Line><Method>_stack_context_handle_exception</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/tornado/web.py</File><Line>1298</Line><Method>wrapper</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/tornadorpc/tornadorpc/base.py</File><Line>259</Line><Method>post</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/tornadorpc/tornadorpc/base.py</File><Line>103</Line><Method>run</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/tornadorpc/tornadorpc/base.py</File><Line>145</Line><Method>dispatch</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/tornadorpc/tornadorpc/utils.py</File><Line>23</Line><Method>getcallargs</Method></Frame></Stack></Post><Post><PostId>20504740</PostId><Url>http://stackoverflow.com/questions/20504740</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./gangnamstyle.py</File><Line>27</Line><Method>csvLineList_2 = csv2[lineCount].split(",")</Method></Frame></Stack></Post><Post><PostId>20513474</PostId><Url>http://stackoverflow.com/questions/20513474</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>F:\My Game\MoonSurvival.py</File><Line>416</Line><Method>module</Method></Frame><Frame><File>F:\My Game\MoonSurvival.py</File><Line>194</Line><Method>run</Method></Frame><Frame><File>F:\My Game\menus.py</File><Line>168</Line><Method>draw</Method></Frame><Frame><File>F:\My Game\menus.py</File><Line>202</Line><Method>renderText</Method></Frame></Stack></Post><Post><PostId>20519901</PostId><Url>http://stackoverflow.com/questions/20519901</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>so.py</File><Line>6</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>20614109</PostId><Url>http://stackoverflow.com/questions/20614109</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>ponto.py</File><Line>79</Line><Method>module</Method></Frame><Frame><File>ponto.py</File><Line>37</Line><Method>identificar_digital</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/pyfprint/pyfprint.py</File><Line>267</Line><Method>identify_finger</Method></Frame></Stack></Post><Post><PostId>20617409</PostId><Url>http://stackoverflow.com/questions/20617409</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>pyshell#18</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>20768095</PostId><Url>http://stackoverflow.com/questions/20768095</Url><Stack><Type>IndexError</Type><Message>too many indices</Message><Frame><File>Python/iw2.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>20820153</PostId><Url>http://stackoverflow.com/questions/20820153</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>ch8.py</File><Line>258</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>20872763</PostId><Url>http://stackoverflow.com/questions/20872763</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\pinky\Downloads\TAREA 3\TAREA 3\Tarea 3\funciones.py</File><Line>50</Line><Method>module</Method></Frame><Frame><File>C:\Users\pinky\Downloads\TAREA 3\TAREA 3\Tarea 3\funciones.py</File><Line>24</Line><Method>generar_rol</Method></Frame></Stack></Post><Post><PostId>20892940</PostId><Url>http://stackoverflow.com/questions/20892940</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>I:\potoh\ProvPy\quichotte.py</File><Line>24</Line><Method>module</Method></Frame><Frame><File>I:\potoh\ProvPy\quichotte.py</File><Line>12</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>21016246</PostId><Url>http://stackoverflow.com/questions/21016246</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Administrator\Desktop\Folder</File><Line>19</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\xlrd\sheet.py</File><Line>395</Line><Method>cell</Method></Frame><Frame><File>C:\Python27\lib\site-packages\xlrd\sheet.py</File><Line>421</Line><Method>cell_xf_index</Method></Frame></Stack></Post><Post><PostId>21058157</PostId><Url>http://stackoverflow.com/questions/21058157</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>D:\Python\raiseException.py</File><Line>51</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21106183</PostId><Url>http://stackoverflow.com/questions/21106183</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>submission.py</File><Line>10</Line><Method>isIn</Method></Frame></Stack></Post><Post><PostId>21202488</PostId><Url>http://stackoverflow.com/questions/21202488</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>main.py</File><Line>41</Line><Method>if check(i) and check(rev(i)):</Method></Frame><Frame><File>main.py</File><Line>30</Line><Method>check</Method></Frame></Stack></Post><Post><PostId>21216732</PostId><Url>http://stackoverflow.com/questions/21216732</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\massi_000\Desktop\COM\Task 1 #2.5.py</File><Line>101</Line><Method>module</Method></Frame><Frame><File>C:\Users\massi_000\Desktop\COM\Task 1 #2.5.py</File><Line>91</Line><Method>main</Method></Frame><Frame><File>C:\Users\massi_000\Desktop\COM\Task 1 #2.5.py</File><Line>80</Line><Method>checkUp</Method></Frame><Frame><File>C:\Users\massi_000\Desktop\COM\Task 1 #2.5.py</File><Line>30</Line><Method>binaryToDenary</Method></Frame></Stack></Post><Post><PostId>21244085</PostId><Url>http://stackoverflow.com/questions/21244085</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\Scripts\test.py</File><Line>9</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21247002</PostId><Url>http://stackoverflow.com/questions/21247002</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>secdepgraph_2.py</File><Line>112</Line><Method>module</Method></Frame><Frame><File>secdepgraph_2.py</File><Line>108</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>21268765</PostId><Url>http://stackoverflow.com/questions/21268765</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#10</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21273346</PostId><Url>http://stackoverflow.com/questions/21273346</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Owner\Downloads\Black Jack Revision.py</File><Line>43</Line><Method>module</Method></Frame><Frame><File>C:\Users\Owner\Downloads\Black Jack Revision.py</File><Line>23</Line><Method>showHand</Method></Frame></Stack></Post><Post><PostId>21326061</PostId><Url>http://stackoverflow.com/questions/21326061</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\My Documents\Python\Lexical analyzer.py</File><Line>62</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21335682</PostId><Url>http://stackoverflow.com/questions/21335682</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Users/Nikolai/Dropbox/NTNU/KTN/WebServer/TCPServer.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21344061</PostId><Url>http://stackoverflow.com/questions/21344061</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>code.py</File><Line>42</Line><Method>module</Method></Frame><Frame><File>code.py</File><Line>17</Line><Method>host=str(sys.argv[1])</Method></Frame></Stack></Post><Post><PostId>21344061</PostId><Url>http://stackoverflow.com/questions/21344061</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>code.py</File><Line>42</Line><Method>module</Method></Frame><Frame><File>code.py</File><Line>17</Line><Method>host=str(sys.argv[1])</Method></Frame></Stack></Post><Post><PostId>21391064</PostId><Url>http://stackoverflow.com/questions/21391064</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\User\Desktop\openFITS.py</File><Line>34</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\pyfits\convenience.py</File><Line>181</Line><Method>getdata</Method></Frame><Frame><File>C:\Python27\lib\site-packages\pyfits\hdu\hdulist.py</File><Line>166</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>21445421</PostId><Url>http://stackoverflow.com/questions/21445421</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/pi/test_files/test_two.py</File><Line>99</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21511420</PostId><Url>http://stackoverflow.com/questions/21511420</Url><Stack><Type>IndexError</Type><Message>index (4) out of range</Message><Frame><File>3.py</File><Line>103</Line><Method>module</Method></Frame><Frame><File>3.py</File><Line>99</Line><Method>main</Method></Frame><Frame><File>3.py</File><Line>46</Line><Method>test</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/scipy/sparse/csr.py</File><Line>281</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/scipy/sparse/csr.py</File><Line>233</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/scipy/sparse/csr.py</File><Line>320</Line><Method>_get_row_slice</Method></Frame></Stack></Post><Post><PostId>21514217</PostId><Url>http://stackoverflow.com/questions/21514217</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\gaming only\Desktop\PyMap\tiling.py</File><Line>30</Line><Method>module</Method></Frame><Frame><File>C:\Users\gaming only\Desktop\PyMap\tiling.py</File><Line>19</Line><Method>tileMap</Method></Frame></Stack></Post><Post><PostId>21538654</PostId><Url>http://stackoverflow.com/questions/21538654</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>/Users/hunterjamesnelson/Documents/bruceArray.py</File><Line>9</Line><Method>module</Method></Frame><Frame><File>/Users/hunterjamesnelson/Documents/bruceArray.py</File><Line>3</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>21565241</PostId><Url>http://stackoverflow.com/questions/21565241</Url><Stack><Type>IndexError</Type><Message>list index out of ran</Message><Frame><File>C:/Users/owner/PycharmProjects/Football TV Guide App/TVGuide.py</File><Line>11</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21578471</PostId><Url>http://stackoverflow.com/questions/21578471</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>G:\Python\Hangman\hangman.py</File><Line>60</Line><Method>module</Method></Frame><Frame><File>G:\Python\Hangman\hangman.py</File><Line>60</Line><Method>listcomp</Method></Frame></Stack></Post><Post><PostId>21618170</PostId><Url>http://stackoverflow.com/questions/21618170</Url><Stack><Type>IndexError</Type><Message>too many indices</Message><Frame><File>C:\Users\gary\Documents\EPSON STUDIES\delaunay.py</File><Line>15</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21756669</PostId><Url>http://stackoverflow.com/questions/21756669</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>myscript.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21768079</PostId><Url>http://stackoverflow.com/questions/21768079</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/andres/Documents/Executables/f2c.py</File><Line>3</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21798086</PostId><Url>http://stackoverflow.com/questions/21798086</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#5</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/Users/Alastair/Desktop/CA.py</File><Line>70</Line><Method>importclues</Method></Frame></Stack></Post><Post><PostId>21899238</PostId><Url>http://stackoverflow.com/questions/21899238</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>sentiment.py</File><Line>23</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21940150</PostId><Url>http://stackoverflow.com/questions/21940150</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>depunctuate.py</File><Line>10</Line><Method>module</Method></Frame><Frame><File>depunctuate.py</File><Line>7</Line><Method>depunctuate</Method></Frame></Stack></Post><Post><PostId>21960586</PostId><Url>http://stackoverflow.com/questions/21960586</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>C:/Users/Dasinator/Documents/Books IX/Python Examples/proj07.py</File><Line>10</Line><Method>module</Method></Frame><Frame><File>C:/Users/Dasinator/Documents/Books IX/Python Examples/proj07.py</File><Line>10</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21960586</PostId><Url>http://stackoverflow.com/questions/21960586</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>C:/Users/Dasinator/Documents/Books IX/Python Examples/proj07.py</File><Line>10</Line><Method>module</Method></Frame><Frame><File>C:/Users/Dasinator/Documents/Books IX/Python Examples/proj07.py</File><Line>10</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21961250</PostId><Url>http://stackoverflow.com/questions/21961250</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>C:/Users/Dasinator/Documents/Books IX/Python Examples/proj07.py</File><Line>10</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>21980783</PostId><Url>http://stackoverflow.com/questions/21980783</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>MoreCrap.py</File><Line>80</Line><Method>module</Method></Frame><Frame><File>MoreCrap.py</File><Line>74</Line><Method>animate</Method></Frame><Frame><File>MoreCrap.py</File><Line>26</Line><Method>position</Method></Frame></Stack></Post><Post><PostId>21997347</PostId><Url>http://stackoverflow.com/questions/21997347</Url><Stack><Type>IndexError</Type><Message>Cannot choose from an empty sequence</Message><Frame><File>C:\Python33\lib\random.py</File><Line>249</Line><Method>choice</Method></Frame><Frame><File>C:\Python33\lib\random.py</File><Line>225</Line><Method>_randbelow</Method></Frame><Frame><File>C:\Users\*******\Desktop\*******\*********Python\new\Final - Copy.py</File><Line>84</Line><Method>module</Method></Frame><Frame><File>C:\Python33\lib\random.py</File><Line>251</Line><Method>choice</Method></Frame></Stack></Post><Post><PostId>21997347</PostId><Url>http://stackoverflow.com/questions/21997347</Url><Stack><Type>IndexError</Type><Message>Cannot choose from an empty sequence</Message><Frame><File>C:\Python33\lib\random.py</File><Line>249</Line><Method>choice</Method></Frame><Frame><File>C:\Python33\lib\random.py</File><Line>225</Line><Method>_randbelow</Method></Frame><Frame><File>C:\Users\*******\Desktop\*******\*********Python\new\Final - Copy.py</File><Line>84</Line><Method>module</Method></Frame><Frame><File>C:\Python33\lib\random.py</File><Line>251</Line><Method>choice</Method></Frame></Stack></Post><Post><PostId>21999433</PostId><Url>http://stackoverflow.com/questions/21999433</Url><Stack><Type>IndexError</Type><Message>pop from empty list</Message><Frame><File>/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/nose/loader.py</File><Line>286</Line><Method>generate</Method></Frame><Frame><File>/Users/olga/workspace-git/matplotlib/lib/matplotlib/testing/decorators.py</File><Line>145</Line><Method>test</Method></Frame><Frame><File>/Users/olga/workspace-git/matplotlib/lib/matplotlib/testing/decorators.py</File><Line>296</Line><Method>_image_directories</Method></Frame></Stack></Post><Post><PostId>22048495</PostId><Url>http://stackoverflow.com/questions/22048495</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python26\ScapyScripts\main.py</File><Line>114</Line><Method>ping2</Method></Frame></Stack></Post><Post><PostId>22049440</PostId><Url>http://stackoverflow.com/questions/22049440</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./erl_testcase.py</File><Line>39</Line><Method>module</Method></Frame><Frame><File>/Library/Python/2.7/site-packages/pyparsing.py</File><Line>317</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>22081239</PostId><Url>http://stackoverflow.com/questions/22081239</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>json2.py</File><Line>42</Line><Method>module</Method></Frame><Frame><File>/Library/Python/2.7/site-packages/pyparsing.py</File><Line>317</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>22087768</PostId><Url>http://stackoverflow.com/questions/22087768</Url><Stack><Type>IndexError</Type><Message>array index out of range</Message><Frame><File>Excel_to_CSV_1.py</File><Line>35</Line><Method>module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/xlrd/sheet.py</File><Line>245</Line><Method>cell</Method></Frame></Stack></Post><Post><PostId>22108184</PostId><Url>http://stackoverflow.com/questions/22108184</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python27\Off5th_Guest_Checkout</File><Line>56</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22108654</PostId><Url>http://stackoverflow.com/questions/22108654</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>E:FileName.py</File><Line>34</Line><Method>module</Method></Frame><Frame><File>E:FileName2.py</File><Line>174</Line><Method>draw</Method></Frame></Stack></Post><Post><PostId>22108909</PostId><Url>http://stackoverflow.com/questions/22108909</Url><Stack><Type>IndexError</Type><Message>too many indices</Message><Frame><File>C:\Users\User\Desktop\Code.py</File><Line>251</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22114760</PostId><Url>http://stackoverflow.com/questions/22114760</Url><Stack><Type>IndexError</Type><Message>0-d arrays can't be indexed.</Message><Frame><File>code5.py</File><Line>40</Line><Method>module</Method></Frame><Frame><File>code5.py</File><Line>36</Line><Method>main</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/sklearn/neighbors/classification.py</File><Line>123</Line><Method>predict</Method></Frame></Stack></Post><Post><PostId>22137382</PostId><Url>http://stackoverflow.com/questions/22137382</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>code.py</File><Line>25</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22140277</PostId><Url>http://stackoverflow.com/questions/22140277</Url><Stack><Type>IndexError</Type><Message>pop from empty list</Message><Frame><File>C:\Python33\trial.py</File><Line>41</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22251945</PostId><Url>http://stackoverflow.com/questions/22251945</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python32\Assignment1.py</File><Line>17</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22289357</PostId><Url>http://stackoverflow.com/questions/22289357</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#187</File><Line>2</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22306390</PostId><Url>http://stackoverflow.com/questions/22306390</Url><Stack><Type>IndexError</Type><Message>list Out of range</Message><Frame><File>Z:\PythonScripts\Make_version_1.py</File><Line>20</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22310612</PostId><Url>http://stackoverflow.com/questions/22310612</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>BackProp.py</File><Line>126</Line><Method>module</Method></Frame><Frame><File>BackProp.py</File><Line>93</Line><Method>TrainEpoch</Method></Frame></Stack></Post><Post><PostId>22360418</PostId><Url>http://stackoverflow.com/questions/22360418</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>E:/Computer Programming/Assignment/check 2.py</File><Line>22</Line><Method>module</Method></Frame><Frame><File>E:/Computer Programming/Assignment/check 2.py</File><Line>22</Line><Method>listcomp</Method></Frame><Frame><File>E:/Computer Programming/Assignment/check 2.py</File><Line>22</Line><Method>listcomp</Method></Frame></Stack></Post><Post><PostId>22393748</PostId><Url>http://stackoverflow.com/questions/22393748</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>E:\MyAi.py</File><Line>27</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22431018</PostId><Url>http://stackoverflow.com/questions/22431018</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>C:\Users\gelalmp\Desktop\Bibha Gelal_SD\testing_Optimization_HWM.py</File><Line>100</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\scipy\optimize\_minimize.py</File><Line>380</Line><Method>minimize</Method></Frame><Frame><File>C:\Python27\lib\site-packages\scipy\optimize\lbfgsb.py</File><Line>314</Line><Method>_minimize_lbfgsb</Method></Frame><Frame><File>C:\Python27\lib\site-packages\scipy\optimize\lbfgsb.py</File><Line>258</Line><Method>func_and_grad</Method></Frame><Frame><File>C:\Users\gelalmp\Desktop\Bibha Gelal_SD\testing_Optimization_HWM.py</File><Line>12</Line><Method>mape</Method></Frame><Frame><File>C:\Users\gelalmp\Desktop\Bibha Gelal_SD\testing_Optimization_HWM.py</File><Line>30</Line><Method>y</Method></Frame></Stack></Post><Post><PostId>22431536</PostId><Url>http://stackoverflow.com/questions/22431536</Url><Stack><Type>IndexError</Type><Message>index 4 is out of bounds for axis 0 with size 3</Message><Frame><File>C:\Users\SEC\Desktop\HDWeathProgram\testing_Optimization_HWM.py</File><Line>135</Line><Method>module</Method></Frame><Frame><File>C:\Python27\lib\site-packages\scipy\optimize\_minimize.py</File><Line>364</Line><Method>minimize</Method></Frame><Frame><File>C:\Python27\lib\site-packages\scipy\optimize\slsqp.py</File><Line>354</Line><Method>_minimize_slsqp</Method></Frame><Frame><File>C:\Python27\lib\site-packages\scipy\optimize\optimize.py</File><Line>261</Line><Method>function_wrapper</Method></Frame><Frame><File>C:\Users\SEC\Desktop\HDWeathProgram\testing_Optimization_HWM.py</File><Line>12</Line><Method>mape</Method></Frame><Frame><File>C:\Users\SEC\Desktop\HDWeathProgram\testing_Optimization_HWM.py</File><Line>30</Line><Method>y</Method></Frame></Stack></Post><Post><PostId>22465558</PostId><Url>http://stackoverflow.com/questions/22465558</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>H:\SVA UK PROGRAM\SVA of UK.py</File><Line>126</Line><Method>module</Method></Frame><Frame><File>H:\SVA UK PROGRAM\SVA of UK.py</File><Line>114</Line><Method>Main</Method></Frame><Frame><File>H:\SVA UK PROGRAM\SVA of UK.py</File><Line>74</Line><Method>CheckDatabase</Method></Frame></Stack></Post><Post><PostId>22534918</PostId><Url>http://stackoverflow.com/questions/22534918</Url><Stack><Type>IndexError</Type><Message>0-d arrays can't be indexed</Message><Frame><File>ucmh.py</File><Line>137</Line><Method>module</Method></Frame><Frame><File>ucmh.py</File><Line>128</Line><Method>main</Method></Frame><Frame><File>ucmh.py</File><Line>119</Line><Method>avg_rho_ucmh_squared</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/scipy/integrate/quadrature.py</File><Line>596</Line><Method>romberg</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/scipy/integrate/quadrature.py</File><Line>477</Line><Method>_difftrap</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/scipy/integrate/quadrature.py</File><Line>94</Line><Method>vfunc</Method></Frame></Stack></Post><Post><PostId>22538478</PostId><Url>http://stackoverflow.com/questions/22538478</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>\\prov-dc\students\jadewusi\crack2.py</File><Line>447</Line><Method>module</Method></Frame><Frame><File>\\prov-dc\students\jadewusi\crack2.py</File><Line>398</Line><Method>main</Method></Frame><Frame><File>\\prov-dc\students\jadewusi\crack2.py</File><Line>253</Line><Method>search_method_3</Method></Frame><Frame><File>\\prov-dc\students\jadewusi\crack2.py</File><Line>206</Line><Method>Cap</Method></Frame></Stack></Post><Post><PostId>22553036</PostId><Url>http://stackoverflow.com/questions/22553036</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\JAVA\test-img\test\test.py</File><Line>12</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22578310</PostId><Url>http://stackoverflow.com/questions/22578310</Url><Stack><Type>IndexError</Type><Message>pop from empty list</Message><Frame><File>project.py</File><Line>70</Line><Method>module</Method></Frame><Frame><File>project.py</File><Line>71</Line><Method>Exchange</Method></Frame></Stack></Post><Post><PostId>22579238</PostId><Url>http://stackoverflow.com/questions/22579238</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>D:/Documents/Scripts/Python/Formatted Print</File><Line>3</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22579321</PostId><Url>http://stackoverflow.com/questions/22579321</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>project.py</File><Line>36</Line><Method>device_connected</Method></Frame></Stack></Post><Post><PostId>22595255</PostId><Url>http://stackoverflow.com/questions/22595255</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>C:\Users\antoniozeus\Desktop\BuyAndHold.py</File><Line>31</Line><Method>na_normalized_price = closep/closep[i-1]</Method></Frame></Stack></Post><Post><PostId>22620505</PostId><Url>http://stackoverflow.com/questions/22620505</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>Z:\group project\QuestionnaireDisplayResults.py</File><Line>69</Line><Method>module</Method></Frame><Frame><File>Z:\group project\QuestionnaireDisplayResults.py</File><Line>64</Line><Method>main</Method></Frame><Frame><File>Z:\group project\QuestionnaireDisplayResults.py</File><Line>14</Line><Method>__init__</Method></Frame><Frame><File>Z:\group project\QuestionnaireDisplayResults.py</File><Line>32</Line><Method>retrieveResponse</Method></Frame></Stack></Post><Post><PostId>22704496</PostId><Url>http://stackoverflow.com/questions/22704496</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>setup.py</File><Line>160</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22724189</PostId><Url>http://stackoverflow.com/questions/22724189</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>db_alerts.py</File><Line>65</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22734890</PostId><Url>http://stackoverflow.com/questions/22734890</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/Users/laszlosandor/anaconda/lib/python2.7/site-packages/gevent/greenlet.py</File><Line>327</Line><Method>run</Method></Frame><Frame><File>/Users/laszlosandor/Downloads/GoogleGeocoding/google_geocoding_LS.py</File><Line>37</Line><Method>workrow</Method></Frame></Stack></Post><Post><PostId>22744576</PostId><Url>http://stackoverflow.com/questions/22744576</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:/Users/Peter/Documents/ARCHIVES/NEW!/Computing/task3version1.0.py</File><Line>20</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22790672</PostId><Url>http://stackoverflow.com/questions/22790672</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>test_parallel.py</File><Line>197</Line><Method>module</Method></Frame><Frame><File>/usr/lib64/python2.6/multiprocessing/pool.py</File><Line>148</Line><Method>map</Method></Frame><Frame><File>/usr/lib64/python2.6/multiprocessing/pool.py</File><Line>422</Line><Method>get</Method></Frame></Stack></Post><Post><PostId>22820986</PostId><Url>http://stackoverflow.com/questions/22820986</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>miner.py</File><Line>12</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22842239</PostId><Url>http://stackoverflow.com/questions/22842239</Url><Stack><Type>IndexError</Type><Message>pop from empty list</Message><Frame><File>pyshell#9</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>22967549</PostId><Url>http://stackoverflow.com/questions/22967549</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>manage.py</File><Line>10</Line><Method>module</Method></Frame><Frame><File>/home/abhishek/projects/studentmptt/local/lib/python2.7/site-packages/django/core/management/__init__.py</File><Line>399</Line><Method>execute_from_command_line</Method></Frame><Frame><File>/home/abhishek/projects/studentmptt/local/lib/python2.7/site-packages/django/core/management/__init__.py</File><Line>392</Line><Method>execute</Method></Frame><Frame><File>/home/abhishek/projects/studentmptt/local/lib/python2.7/site-packages/django/core/management/base.py</File><Line>242</Line><Method>run_from_argv</Method></Frame><Frame><File>/home/abhishek/projects/studentmptt/local/lib/python2.7/site-packages/django/core/management/base.py</File><Line>285</Line><Method>execute</Method></Frame><Frame><File>/home/abhishek/projects/studentmptt/local/lib/python2.7/site-packages/django/core/management/base.py</File><Line>415</Line><Method>handle</Method></Frame><Frame><File>/home/abhishek/projects/studentmptt/local/lib/python2.7/site-packages/django/core/management/commands/shell.py</File><Line>83</Line><Method>handle_noargs</Method></Frame><Frame><File>/home/abhishek/projects/studentmptt/studentsite/code.py</File><Line>12</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23003141</PostId><Url>http://stackoverflow.com/questions/23003141</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Python27/Maze Assignment MAZE1m.py</File><Line>75</Line><Method>module</Method></Frame><Frame><File>C:/Python27/Maze Assignment MAZE1m.py</File><Line>61</Line><Method>solve_maze</Method></Frame><Frame><File>C:/Python27/Maze Assignment  MAZE1m.py</File><Line>46</Line><Method>flood_fill</Method></Frame></Stack></Post><Post><PostId>23030295</PostId><Url>http://stackoverflow.com/questions/23030295</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>python</File><Line>27</Line><Method>module</Method></Frame><Frame><File>python</File><Line>7</Line><Method>censor</Method></Frame></Stack></Post><Post><PostId>23039524</PostId><Url>http://stackoverflow.com/questions/23039524</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>historical_tempo1.py</File><Line>23</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23058346</PostId><Url>http://stackoverflow.com/questions/23058346</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#136</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23089881</PostId><Url>http://stackoverflow.com/questions/23089881</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>test.py</File><Line>63</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23122248</PostId><Url>http://stackoverflow.com/questions/23122248</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\workspace-php\test\pdfminer\pdfminer\tests.py</File><Line>13</Line><Method>setUp</Method></Frame><Frame><File>C:\workspace-php\test\pdfminer\pdfminer\tests.py</File><Line>13</Line><Method>setUp</Method></Frame></Stack></Post><Post><PostId>23122248</PostId><Url>http://stackoverflow.com/questions/23122248</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\workspace-php\test\pdfminer\pdfminer\tests.py</File><Line>13</Line><Method>setUp</Method></Frame><Frame><File>C:\workspace-php\test\pdfminer\pdfminer\tests.py</File><Line>13</Line><Method>setUp</Method></Frame></Stack></Post><Post><PostId>23123094</PostId><Url>http://stackoverflow.com/questions/23123094</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Users/******/Documents/MalwareTycoon/main.py</File><Line>151</Line><Method>generate_network</Method></Frame></Stack></Post><Post><PostId>23198847</PostId><Url>http://stackoverflow.com/questions/23198847</Url><Stack><Type>IndexError</Type><Message>bytearray index out of range</Message><Frame><File>test.py</File><Line>62</Line><Method>module  # The line 62 is the location of x[0]=datosOEM[k]</Method></Frame></Stack></Post><Post><PostId>23212058</PostId><Url>http://stackoverflow.com/questions/23212058</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>spoj.py</File><Line>38</Line><Method>module</Method></Frame><Frame><File>spoj.py</File><Line>15</Line><Method>resolve</Method></Frame></Stack></Post><Post><PostId>23296118</PostId><Url>http://stackoverflow.com/questions/23296118</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Or\define\projects\rss feed\the progect\test.py</File><Line>33</Line><Method>module</Method></Frame><Frame><File>C:\Or\define\projects\rss feed\the progect\test.py</File><Line>14</Line><Method>update</Method></Frame></Stack></Post><Post><PostId>23301861</PostId><Url>http://stackoverflow.com/questions/23301861</Url><Stack><Type>IndexError</Type><Message>index out of range for array</Message><Frame><File>temp.py</File><Line>16</Line><Method>module</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/scipy/stats/distributions.py</File><Line>5637</Line><Method>__init__</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/numpy/core/fromnumeric.py</File><Line>103</Line><Method>take</Method></Frame></Stack></Post><Post><PostId>23366175</PostId><Url>http://stackoverflow.com/questions/23366175</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#12</File><Line>7</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23387413</PostId><Url>http://stackoverflow.com/questions/23387413</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Masyh\Desktop\autowin_test.py</File><Line>8</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23449945</PostId><Url>http://stackoverflow.com/questions/23449945</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>E:\Geekster_Bot\Geekster_Bot Code\Geekster_Bot_Test_Write</File><Line>54</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23547382</PostId><Url>http://stackoverflow.com/questions/23547382</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>CFSR2SWAN.py</File><Line>52</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23773220</PostId><Url>http://stackoverflow.com/questions/23773220</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>ncall.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23773220</PostId><Url>http://stackoverflow.com/questions/23773220</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>ncall.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23773220</PostId><Url>http://stackoverflow.com/questions/23773220</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>ncall.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23773220</PostId><Url>http://stackoverflow.com/questions/23773220</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>ncall.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>ncall.py</File><Line>15</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23794326</PostId><Url>http://stackoverflow.com/questions/23794326</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Python27/practice example/datascraper templates.py</File><Line>21</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23836682</PostId><Url>http://stackoverflow.com/questions/23836682</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>G:\Lvl 5\sofware\menuNEWex - Copy.py</File><Line>62</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>23853828</PostId><Url>http://stackoverflow.com/questions/23853828</Url><Stack><Type>IndexError</Type><Message>index 7 is out of bounds for axis 1 with size 7</Message><Frame><File>./gensim_topic_modeling.py</File><Line>98</Line><Method>module</Method></Frame><Frame><File>/Users/me/anaconda/lib/python2.7/site-packages/gensim/models/ldamodel.py</File><Line>306</Line><Method>__init__</Method></Frame><Frame><File>/Users/me/anaconda/lib/python2.7/site-packages/gensim/models/ldamodel.py</File><Line>543</Line><Method>update</Method></Frame><Frame><File>/Users/me/anaconda/lib/python2.7/site-packages/gensim/models/ldamodel.py</File><Line>454</Line><Method>log_perplexity</Method></Frame><Frame><File>/Users/me/anaconda/lib/python2.7/site-packages/gensim/models/ldamodel.py</File><Line>630</Line><Method>bound</Method></Frame><Frame><File>/Users/me/anaconda/lib/python2.7/site-packages/gensim/models/ldamodel.py</File><Line>366</Line><Method>inference</Method></Frame></Stack></Post><Post><PostId>23883575</PostId><Url>http://stackoverflow.com/questions/23883575</Url><Stack><Type>IndexError</Type><Message>invalid index to scalar variable.</Message><Frame><File>missing_sort.py</File><Line>16</Line><Method>start="%s%s%s%s%s" % (data.index[0][7:11]</Method></Frame></Stack></Post><Post><PostId>23901687</PostId><Url>http://stackoverflow.com/questions/23901687</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/usr/lib/python2.7/dist-packages/pip/basecommand.py</File><Line>122</Line><Method>main</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/pip/commands/install.py</File><Line>278</Line><Method>run</Method></Frame><Frame><File>/usr/lib/python2.7/dist-packages/pip/req.py</File><Line>1259</Line><Method>prepare_files</Method></Frame></Stack></Post><Post><PostId>24011799</PostId><Url>http://stackoverflow.com/questions/24011799</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>test.py</File><Line>11</Line><Method>module</Method></Frame><Frame><File>test.py</File><Line>26</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24011799</PostId><Url>http://stackoverflow.com/questions/24011799</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>test.py</File><Line>11</Line><Method>module</Method></Frame><Frame><File>test.py</File><Line>26</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24018897</PostId><Url>http://stackoverflow.com/questions/24018897</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>addlists.py</File><Line>101</Line><Method>module</Method></Frame><Frame><File>addlists.py</File><Line>13</Line><Method>wrapper</Method></Frame><Frame><File>addlists.py</File><Line>97</Line><Method>c</Method></Frame><Frame><File>addlists.py</File><Line>82</Line><Method>lmerge</Method></Frame></Stack></Post><Post><PostId>24056523</PostId><Url>http://stackoverflow.com/questions/24056523</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#105</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:/Users/prince/Desktop/CsvRead.py</File><Line>55</Line><Method>ParseData</Method></Frame></Stack></Post><Post><PostId>24106605</PostId><Url>http://stackoverflow.com/questions/24106605</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/usr/lib/python2.7/site-packages/IPython/core/ultratb.py</File><Line>760</Line><Method>structured_traceback</Method></Frame><Frame><File>/usr/lib/python2.7/site-packages/IPython/core/ultratb.py</File><Line>242</Line><Method>_fixed_getinnerframes</Method></Frame><Frame><File>/usr/lib64/python2.7/inspect.py</File><Line>1043</Line><Method>getinnerframes</Method></Frame><Frame><File>/usr/lib64/python2.7/inspect.py</File><Line>1007</Line><Method>getframeinfo</Method></Frame><Frame><File>/usr/lib64/python2.7/inspect.py</File><Line>580</Line><Method>findsource</Method></Frame></Stack></Post><Post><PostId>24112445</PostId><Url>http://stackoverflow.com/questions/24112445</Url><Stack><Type>IndexError</Type><Message>invalid index</Message><Frame><File>sublimePygame</File><Line>29</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24123027</PostId><Url>http://stackoverflow.com/questions/24123027</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./test.py</File><Line>7</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24139263</PostId><Url>http://stackoverflow.com/questions/24139263</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#11</File><Line>1</Line><Method>module</Method></Frame><Frame><File>pyshell#10</File><Line>8</Line><Method>third_largest</Method></Frame></Stack></Post><Post><PostId>24192254</PostId><Url>http://stackoverflow.com/questions/24192254</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>lambda</Method></Frame></Stack></Post><Post><PostId>24247872</PostId><Url>http://stackoverflow.com/questions/24247872</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#3</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24254525</PostId><Url>http://stackoverflow.com/questions/24254525</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>twitterBot2.py</File><Line>78</Line><Method>module</Method></Frame><Frame><File>twitterBot2.py</File><Line>42</Line><Method>getUserName</Method></Frame></Stack></Post><Post><PostId>24278627</PostId><Url>http://stackoverflow.com/questions/24278627</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>/usr/lib/python2.7/runpy.py</File><Line>162</Line><Method>_run_module_as_main</Method></Frame><Frame><File>/usr/lib/python2.7/runpy.py</File><Line>72</Line><Method>_run_code</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/JCC-2.19-py2.7-linux-x86_64.egg/jcc/__main__.py</File><Line>107</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/JCC-2.19-py2.7-linux-x86_64.egg/jcc/cpp.py</File><Line>687</Line><Method>jcc</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/JCC-2.19-py2.7-linux-x86_64.egg/jcc/cpp.py</File><Line>1174</Line><Method>code</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/JCC-2.19-py2.7-linux-x86_64.egg/jcc/cpp.py</File><Line>310</Line><Method>find_method</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/JCC-2.19-py2.7-linux-x86_64.egg/jcc/cpp.py</File><Line>51</Line><Method>getJavaException</Method></Frame></Stack></Post><Post><PostId>24312579</PostId><Url>http://stackoverflow.com/questions/24312579</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>configure-ng.py</File><Line>2816</Line><Method>module</Method></Frame><Frame><File>configure-ng.py</File><Line>2780</Line><Method>main</Method></Frame><Frame><File>configure-ng.py</File><Line>695</Line><Method>from_introspection</Method></Frame></Stack></Post><Post><PostId>24315493</PostId><Url>http://stackoverflow.com/questions/24315493</Url><Stack><Type>IndexError</Type><Message>index out of bounds</Message><Frame><File>test.py</File><Line>203</Line><Method>module</Method></Frame><Frame><File>test.py</File><Line>71</Line><Method>main</Method></Frame><Frame><File>test.py</File><Line>137</Line><Method>get_foreground_counts</Method></Frame></Stack></Post><Post><PostId>24333382</PostId><Url>http://stackoverflow.com/questions/24333382</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>C:\Users\T\Desktop\pickle_process\pickle_process.py</File><Line>16</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24426131</PostId><Url>http://stackoverflow.com/questions/24426131</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>transpose.py</File><Line>66</Line><Method>module</Method></Frame><Frame><File>transpose.py</File><Line>6</Line><Method>main</Method></Frame><Frame><File>transpose.py</File><Line>27</Line><Method>readinput</Method></Frame></Stack></Post><Post><PostId>24489235</PostId><Url>http://stackoverflow.com/questions/24489235</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:\Python25\PYGAME\TileRpg\LevelEditorMain.py</File><Line>7</Line><Method>module</Method></Frame><Frame><File>C:\Python25\PYGAME\TileRpg\LevelEditorMain.py</File><Line>17</Line><Method>Main</Method></Frame><Frame><File>C:\Python25\PYGAME\TileRpg\Map.py</File><Line>48</Line><Method>loadMap</Method></Frame></Stack></Post><Post><PostId>24553616</PostId><Url>http://stackoverflow.com/questions/24553616</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/Users/-----/PycharmProjects/First/test.py</File><Line>20</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24591798</PostId><Url>http://stackoverflow.com/questions/24591798</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>twitterBot3.py</File><Line>111</Line><Method>module</Method></Frame><Frame><File>twitterBot3.py</File><Line>86</Line><Method>get_tweet_meta</Method></Frame></Stack></Post><Post><PostId>24594420</PostId><Url>http://stackoverflow.com/questions/24594420</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>J:\Desktop\urlformstringtest.py</File><Line>17</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24602231</PostId><Url>http://stackoverflow.com/questions/24602231</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/mint/PycharmProjects/DnD_Project/Power_Name.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24623734</PostId><Url>http://stackoverflow.com/questions/24623734</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>nitro_catagories.py</File><Line>30</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24680099</PostId><Url>http://stackoverflow.com/questions/24680099</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>E:\Python\RBTrends\RBTrendsAnalysis.py</File><Line>55</Line><Method>module</Method></Frame><Frame><File>E:\Python\RBTrends\RBTrendsAnalysis.py</File><Line>32</Line><Method>createArticles</Method></Frame></Stack></Post><Post><PostId>24734731</PostId><Url>http://stackoverflow.com/questions/24734731</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:/Python34/LEARN/play/dapatkan_index2.py</File><Line>7</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24752426</PostId><Url>http://stackoverflow.com/questions/24752426</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./5</File><Line>10</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24791666</PostId><Url>http://stackoverflow.com/questions/24791666</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>C:\Users\Charlie Chiang\Desktop\9G.py</File><Line>17</Line><Method>module</Method></Frame><Frame><File>C:\Users\Charlie Chiang\Desktop\9G.py</File><Line>10</Line><Method>countLetters</Method></Frame></Stack></Post><Post><PostId>24803349</PostId><Url>http://stackoverflow.com/questions/24803349</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Users\Kenny\Desktop\TTUMExcel.py</File><Line>26</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>24809450</PostId><Url>http://stackoverflow.com/questions/24809450</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>./align.py</File><Line>316</Line><Method>module</Method></Frame><Frame><File>./align.py</File><Line>146</Line><Method>readAlignedMLF</Method></Frame></Stack></Post><Post><PostId>24821165</PostId><Url>http://stackoverflow.com/questions/24821165</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>G:/P/14. Digital metrics - Phase 2/3. Execution/4. Code/Code Final/Unmetric Post level/test.py</File><Line>66</Line><Method>module</Method></Frame><Frame><File>G:/P/14. Digital metrics - Phase 2/3. Execution/4. Code/Code Final/Unmetric Post level/test.py</File><Line>53</Line><Method>FBPostupload</Method></Frame></Stack></Post><Post><PostId>24847687</PostId><Url>http://stackoverflow.com/questions/24847687</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>./author_preprocessing.py</File><Line>14</Line><Method>module</Method></Frame><Frame><File>build/bdist.linux-x86_64/egg/name_tools/match.py</File><Line>102</Line><Method>match</Method></Frame><Frame><File>build/bdist.linux-x86_64/egg/name_tools/match.py</File><Line>19</Line><Method>middle_initials</Method></Frame></Stack></Post><Post><PostId>24848015</PostId><Url>http://stackoverflow.com/questions/24848015</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\Python32\Space Invaders.py</File><Line>151</Line><Method>module</Method></Frame><Frame><File>C:\Python32\Space Invaders.py</File><Line>98</Line><Method>update</Method></Frame><Frame><File>C:\Python32\Space Invaders.py</File><Line>86</Line><Method>hit_enemy</Method></Frame></Stack></Post><Post><PostId>24890643</PostId><Url>http://stackoverflow.com/questions/24890643</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/Spark/spark-1.0.1-bin-hadoop2/python/pyspark/worker.py</File><Line>77</Line><Method>main</Method></Frame><Frame><File>/home/Spark/spark-1.0.1-bin-hadoop2/python/pyspark/rdd.py</File><Line>283</Line><Method>func</Method></Frame><Frame><File>/home/Spark/spark-1.0.1-bin-hadoop2/python/pyspark/rdd.py</File><Line>1134</Line><Method>_mergeCombiners</Method></Frame><Frame><File>/home/Programming/PySpark-Affinities/cartesian.py</File><Line>60</Line><Method>pairwise_blocks</Method></Frame></Stack></Post><Post><PostId>24906787</PostId><Url>http://stackoverflow.com/questions/24906787</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#12</File><Line>1</Line><Method>module</Method></Frame><Frame><File>C:/Users/Yifan/Desktop/data/Training data/Method2.py</File><Line>10</Line><Method>method2</Method></Frame></Stack></Post><Post><PostId>25065645</PostId><Url>http://stackoverflow.com/questions/25065645</Url><Stack><Type>IndexError</Type><Message>Layer [Padding] not found</Message><Frame><File>rewrite_pcap.py</File><Line>18</Line><Method>module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/scapy/packet.py</File><Line>758</Line><Method>__getitem__</Method></Frame></Stack></Post><Post><PostId>25113814</PostId><Url>http://stackoverflow.com/questions/25113814</Url><Stack><Type>IndexError</Type><Message>index (22) out of range (0</Message><Frame><File>E:/genetic image/genetic_image.py</File><Line>106</Line><Method>module</Method></Frame><Frame><File>E:/genetic image/genetic_image.py</File><Line>93</Line><Method>generate</Method></Frame><Frame><File>E:/genetic image/genetic_image.py</File><Line>41</Line><Method>seed_test</Method></Frame></Stack></Post><Post><PostId>25125300</PostId><Url>http://stackoverflow.com/questions/25125300</Url><Stack><Type>IndexError</Type><Message>too many indices</Message><Frame><File>C:\Users\app\Documents\Python Scripts\gbc_classifier_test.py</File><Line>150</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25267422</PostId><Url>http://stackoverflow.com/questions/25267422</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>~/myscript.py</File><Line>122</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25294224</PostId><Url>http://stackoverflow.com/questions/25294224</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>pyshell#394</File><Line>1</Line><Method>module</Method></Frame><Frame><File>pyshell#389</File><Line>12</Line><Method>meth3</Method></Frame></Stack></Post><Post><PostId>25322387</PostId><Url>http://stackoverflow.com/questions/25322387</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pdb_search.py</File><Line>3</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25344092</PostId><Url>http://stackoverflow.com/questions/25344092</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>1535</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>1529</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>1278</Line><Method>default_dispatcher</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>1102</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>572</Line><Method>dispatch</Method></Frame><Frame><File>/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py</File><Line>570</Line><Method>dispatch</Method></Frame><Frame><File>/base/data/home/apps/s~prescuem/1.378016087775419183/main.py</File><Line>124</Line><Method>post</Method></Frame></Stack></Post><Post><PostId>25344612</PostId><Url>http://stackoverflow.com/questions/25344612</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>D:\Programming\ProjectEuler\p27.py</File><Line>33</Line><Method>module</Method></Frame><Frame><File>D:\Programming\ProjectEuler\p27.py</File><Line>20</Line><Method>isPrime</Method></Frame></Stack></Post><Post><PostId>25385414</PostId><Url>http://stackoverflow.com/questions/25385414</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>columnstrip.py</File><Line>4</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25407328</PostId><Url>http://stackoverflow.com/questions/25407328</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>C:\Users\ACER\Documents\works\source code\python practice\Test1.py</File><Line>4</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25417612</PostId><Url>http://stackoverflow.com/questions/25417612</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>main.py</File><Line>75</Line><Method>module</Method></Frame><Frame><File>main.py</File><Line>32</Line><Method>main</Method></Frame></Stack></Post><Post><PostId>25443564</PostId><Url>http://stackoverflow.com/questions/25443564</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>/usr/local/bin/buildozer</File><Line>9</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/buildozer-0.16_dev-py2.7.egg/buildozer/scripts/client.py</File><Line>13</Line><Method>main</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/buildozer-0.16_dev-py2.7.egg/buildozer/__init__.py</File><Line>959</Line><Method>run_command</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/buildozer-0.16_dev-py2.7.egg/buildozer/target.py</File><Line>85</Line><Method>run_commands</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/buildozer-0.16_dev-py2.7.egg/buildozer/target.py</File><Line>97</Line><Method>cmd_debug</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/buildozer-0.16_dev-py2.7.egg/buildozer/__init__.py</File><Line>193</Line><Method>build</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/buildozer-0.16_dev-py2.7.egg/buildozer/targets/android.py</File><Line>467</Line><Method>build_package</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/buildozer-0.16_dev-py2.7.egg/buildozer/__init__.py</File><Line>656</Line><Method>get_version</Method></Frame></Stack></Post><Post><PostId>25516830</PostId><Url>http://stackoverflow.com/questions/25516830</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/rob/Applications/Ambient/AmbientDecode_02.py</File><Line>82</Line><Method>millis = int(row[1])</Method></Frame></Stack></Post><Post><PostId>25538006</PostId><Url>http://stackoverflow.com/questions/25538006</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>playlist.py</File><Line>34</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25558844</PostId><Url>http://stackoverflow.com/questions/25558844</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>pyshell#4</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25665285</PostId><Url>http://stackoverflow.com/questions/25665285</Url><Stack><Type>IndexError</Type><Message>tuple index out of range</Message><Frame><File>plot_data.py</File><Line>7</Line><Method>module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/pyplot.py</File><Line>1894</Line><Method>plotfile</Method></Frame><Frame><File>/usr/lib/pymodules/python2.7/matplotlib/pyplot.py</File><Line>1889</Line><Method>getname_val</Method></Frame></Stack></Post><Post><PostId>25671059</PostId><Url>http://stackoverflow.com/questions/25671059</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>pyshell#25</File><Line>1</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25689483</PostId><Url>http://stackoverflow.com/questions/25689483</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>aligner.py</File><Line>30</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25695663</PostId><Url>http://stackoverflow.com/questions/25695663</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>C:\python_scripts\geocoder_new\geocoder\example_input_and_output\geocoder.py</File><Line>19</Line><Method>module</Method></Frame><Frame><File>C:\python_scripts\geocoder_new\geocoder\example_input_and_output\geocoder.py</File><Line>8</Line><Method>geocode</Method></Frame></Stack></Post><Post><PostId>25699983</PostId><Url>http://stackoverflow.com/questions/25699983</Url><Stack><Type>IndexError</Type><Message>string index out of range</Message><Frame><File>test.py</File><Line>6</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25715298</PostId><Url>http://stackoverflow.com/questions/25715298</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/home/peter/PycharmProjects/Derv_Algo/calc.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25733454</PostId><Url>http://stackoverflow.com/questions/25733454</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>/root/Documents/python/pci3</File><Line>22</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25744059</PostId><Url>http://stackoverflow.com/questions/25744059</Url><Stack><Type>IndexError</Type><Message>list assignment index out of range</Message><Frame><File>/home/nkumar/workplacekepler/random/src/limit.py</File><Line>26</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>25812363</PostId><Url>http://stackoverflow.com/questions/25812363</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>collatz.py</File><Line>21</Line><Method>module</Method></Frame><Frame><File>collatz.py</File><Line>9</Line><Method>collatz_length</Method></Frame></Stack></Post><Post><PostId>25829331</PostId><Url>http://stackoverflow.com/questions/25829331</Url><Stack><Type>IndexError</Type><Message>list index out of range</Message><Frame><File>ebooks.py</File><Line>79</Line><Method>module</Method></Frame><Frame><File>ebooks.py</File><Line>51</Line><Method>grab_tweets</Method></Frame></Stack></Post></Posts>